\documentclass{pnastwo}
\usepackage{cite}
\usepackage{color}
\usepackage{pgf}
\usepackage[normalem]{ulem}
\newcommand{\jri}[1]{\textcolor{red}{\scriptsize #1}}
\newcommand{\citex}{\textcolor{red}{\bf CITE}}
\newcommand{\X}{\textcolor{red}{\bf X}}
\usepackage{multibib}
\RequirePackage[normalem]{ulem} \RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} \providecommand{\DIFadd}[1]{{\protect\color{blue}\uwave{#1}}} \providecommand{\DIFdel}[1]{{\protect\color{red}\sout{#1}}}                      \providecommand{\DIFaddbegin}{} \providecommand{\DIFaddend}{} \providecommand{\DIFdelbegin}{} \providecommand{\DIFdelend}{} \providecommand{\DIFaddFL}[1]{\DIFadd{#1}} \providecommand{\DIFdelFL}[1]{\DIFdel{#1}} \providecommand{\DIFaddbeginFL}{} \providecommand{\DIFaddendFL}{} \providecommand{\DIFdelbeginFL}{} \providecommand{\DIFdelendFL}{} 
\begin{document}


\title{Demography and linked selection in wild and domesticated maize}
\author{Timothy M. Beissinger\affil{1}{Dept. of Plant Sciences, University of
    California, Davis, CA, USA}\affil{2}{US Department of Agriculture, Agricultural Research Service, Columbia, MO, USA}\affil{3}{Division of Plant Sciences, University of Missouri, Columbia, MO, USA} Li Wang\affil{4}{Iowa State University, Ames, IA, USA}, Kate Crosby\affil{1}{}, Arun
  Durvasula\affil{1}{}, Matthew Hufford\affil{4}{}, \and Jeffrey
  Ross-Ibarra\affil{1}{}\affil{5}{Genome Center and Center for population biology, University of
    California, Davis, CA, USA} }

\DIFdelbegin \DIFdelend \DIFaddbegin \significancetext{Both selection and demographic change play important roles in shaping diversity across the genome, but clear empirical examples of the interplay of these two forces are lacking.
Here we document the combined effects of demography and linked selection on genome-wide diversity in domesticated maize and its wild ancestor teosinte.  We estimate that maize underwent a bottleneck of 5\% of the size of the ancestral teosinte population, but that recent expansion has resulted in a maize population perhaps orders of magnitude larger than teosinte.  We show that positive selection on new mutations has had relatively little effect on genetic diversity, but that selection against deleterious mutations has dramatically reduced diversity in and immediately around genes in both taxa.  We find that the relative effect of selection depend qualitatively on the age of the polymorphisms evaluated: while older polymorphisms in maize show more limited effects of linked selection, new mutations instead reflect its larger current size and more efficient purifying selection. Our results demonstrate that a complete understanding of genome-wide patterns of diversity will require careful assessment of both demographic history and the effects of linked selection. }
\DIFaddend 

\maketitle

\begin{article}

\begin{abstract}
  Unique selective and demographic \DIFaddbegin \DIFadd{forces }\DIFaddend operate on domesticated plant species. These forces interact during and after domestication to generate the patterns of DNA variability that are persistent \DIFdelbegin \DIFdel{in crop species }\DIFdelend today. To quantify the interplay between demography and selection, we investigated genetic diversity in maize, one of the most important crops for food, feed, and fuel world-wide. \DIFdelbegin \DIFdel{We utilized }\DIFdelend \DIFaddbegin \DIFadd{Our sample included }\DIFaddend whole genome sequence data from 23 maize and 13 teosinte individuals\DIFdelbegin \DIFdel{to make inferences}\DIFdelend .  We obtained a complete estimate of the population size fluctuations and other demographic parameters experienced by maize as it was domesticated from teosinte. Here, we show that maize went through a domestication bottleneck with a population size of approximately 5\% that of teosinte before it experienced rapid population size expansion post-domestication. We observe that hard sweeps \DIFdelbegin \DIFdel{, specifically positive selection on new genic mutations , }\DIFdelend \DIFaddbegin \DIFadd{on genic mutations }\DIFaddend are not the primary force driving maize evolution. We find that a reduced population size during domestication decreased the efficiency of purifying selection to purge deleterious alleles from maize. However, expansion after domestication has since increased the efficiency of purifying selection to levels \DIFdelbegin \DIFdel{superior to }\DIFdelend \DIFaddbegin \DIFadd{exceeding }\DIFaddend those seen in teosinte. Our results demonstrate that \DIFaddbegin \DIFadd{particularly }\DIFaddend in domesticated species \DIFaddbegin \DIFadd{or bottlenecked species}\DIFaddend , demographic and selective history in the ancient and recent past both contribute to genetic variability that is present today, providing substantial implications for the continued improvement of domesticated species.

\end{abstract}

\DIFaddbegin \jri{we need a starting paragraph on linked selection and demography. check out slotte 2014 briefings functional genomics for a great review of ideas. then we do a paragraph on why domesticates are great for this (modified version of your below)}

\DIFaddend \dropcap{D}omesticated plant species evolve in a unique fashion
compared to their wild counterparts \cite{doebley2006}. This
is a result of both the anthropomorphic nature of artificial selection on
domesticates \cite{purugganan2009} as well as the demographic characteristics of the domestication
bottleneck(s) that they tend to have experienced
\cite{ross2007}. However, the
complex interplay between selective pressures and demographic
limitations, and the impact that this interplay has on identifying
selection and understanding demography, is not fully understood. Although a large body of
research that involves searching the genomes of domesticated species for evidence
of positive selection exists \cite{hufford2012, he2011, vigouroux2002, chapman2008}, these studies tend to focus on
identifying or mapping particular genes or regions that play an
important role in phenotypic evolution. In contrast, knowledge regarding the impacts that demography and
selection have on whole-genome patterns of genetic variability remains limited.


\DIFdelbegin \DIFdel{Maize represents an excellent organism to study these
phenomena. Maize is a species of tremendous importance worldwide as
both a staple crop \mbox{\cite{shiferaw2011} }and as a model for
understanding plant evolution \mbox{\cite{strable2009}}. }\DIFdelend Broadly speaking, archaeological and genetic studies have
established that maize domestication is likely to have taken place in
 Mexico approximately 9,000 years bp
\cite{smith1995,matsuoka2002}. Teosinte, the most
recent wild ancestor to maize, remains extant throughout much of the
Americas \cite{wilkes1967}. Additionally, several large-effect
domestication loci \cite{doebley1995, wills2013, wang2015} and putative domestication
regions \cite{hufford2012} have been identified. But despite all that is
known about maize domestication, the parameters of the
domestication process remain uncertain. Specifically, the size of the
maize domestication bottleneck has not been estimated independently of
the bottleneck's duration, nor are there sequence-based estimates of the effective
population size of modern maize. Sequence information from maize and
teosinte plants may therefore be utilized to address these questions.

To that end, the objectives of our study were to 1) investigate the
relative importance of different forms of selection on whole-genome
variability in both maize and teosinte 2) research the impact that the
domestication process has had on genetic variability in maize, and how
this compares to the impact of a different demographic history in
teosinte; and 3) precisely estimate the parameters of the
maize domestication bottleneck.  \cite{chia2012}.

\section{Results}
\DIFdelbegin \subsection{\DIFdel{Patterns of diversity differ between genic and  non-genic regions of the genome}}
\addtocounter{subsection}{-1}\DIFdelend To investigate how demography and linked selection have shaped patterns of diversity in maize and teosinte, we \DIFdelbegin \DIFdel{reanalyzed }\DIFdelend \DIFaddbegin \DIFadd{analyzed }\DIFaddend data from 23 maize and 13 teosinte genomes from the maize HapMap 2 \DIFdelbegin \DIFdel{project \mbox{\cite{chia2012} }}\DIFdelend \DIFaddbegin \DIFadd{and HapMap 3 projects \mbox{\cite{chia2012} }}\jri{add cite to HM3}\DIFaddend .
We find broad differences in genic and \DIFdelbegin \DIFdel{non-genic }\DIFdelend \DIFaddbegin \DIFadd{intergenic }\DIFaddend diversity consistent with earlier results  \cite{hufford2012}(Figure \ref{fig:diversity})\DIFdelbegin \DIFdel{.  
In }\DIFdelend \DIFaddbegin \DIFadd{:  }\jri{pi graph should use greek letter $\pi$} \DIFadd{in }\DIFaddend maize, mean pairwise diversity ($\pi$) within genes was significantly lower than at \DIFdelbegin \DIFdel{positions }\DIFdelend \DIFaddbegin \DIFadd{site }\DIFaddend at least 5 kb away from genes (0.00668 vs 0.00691, $p<2\times 10^{-44}$). 
Diversity differences in teosinte are even more pronounced (0.0088 vs. 0.0115, $p\approx 0$). 
Differences were also apparent in the site frequency spectrum, with mean Tajima's D positive in genic regions in both maize (0.4) and teosinte (0.013) but negative outside of genes (-0.087 in maize and -0.25 in teosinte, $p\approx 0$ for both comparisons).
These observations suggest that diversity in genes is not evolving neutrally, but instead is reduced by the impacts of selection on linked sites. 

\begin{figure}
\begin{center}
  \DIFdelbeginFL \DIFdelendFL \DIFaddbeginFL \includegraphics[width=.4\textwidth] \DIFaddendFL {FigsAndFiles/Pi_and_Tajima.png}
\end{center}
\caption{Pairwise diversity \DIFaddbeginFL \DIFaddFL{$\pi$ }\DIFaddendFL (\DIFdelbeginFL \DIFdelFL{Pi}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{A,B}\DIFaddendFL ) and Tajimas D \DIFdelbeginFL \DIFdelFL{were calculated inside }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{(C,D) in 1kb windows from genic }\DIFaddendFL and \DIFdelbeginFL \DIFdelFL{outside }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{nongenic regions }\DIFaddendFL of \DIFdelbeginFL \DIFdelFL{genes for }\DIFdelendFL maize \DIFaddbeginFL \DIFaddFL{(A,C) }\DIFaddendFL and teosinte \DIFdelbeginFL \DIFdelFL{. In both taxa}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{(B}\DIFaddendFL ,\DIFdelbeginFL \DIFdelFL{a dearth of rare alleles was observed within genes relative to outside of genes, which led to higher values of Tajima's }\DIFdelendFL D\DIFdelbeginFL \DIFdelFL{outside of genes than inside}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{)}\DIFaddendFL . \DIFdelbeginFL \DIFdelendFL \DIFaddbeginFL \DIFaddFL{Shown in }\DIFaddendFL A \DIFdelbeginFL \DIFdelFL{:}\DIFdelFL{Mean }\DIFdelendFL and \DIFdelbeginFL \DIFdelFL{standard deviation of pi for 1 kb genic and non-genic windows of maize; }\DIFdelendFL B \DIFdelbeginFL \DIFdelFL{:}\DIFdelFL{Mean }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{are means }\DIFaddendFL and \DIFaddbeginFL \DIFaddFL{one }\DIFaddendFL standard deviation\DIFdelbeginFL \DIFdelFL{of pi for 1 kb genic and non-genic windows of teosinte  }\DIFdelFL{C:}\DIFdelFL{Histogram of Tajima's D in 1 kb windows in maize; }\DIFdelFL{D:}\DIFdelFL{Histogram of Tajima's D in 1 kb windows in teosinte}\DIFdelendFL .   \label{fig:diversity} }
\end{figure}

\subsection{Hard sweeps do not explain diversity differences} Selection acting to increase the frequency of a new beneficial mutation will leave a signature of reduced diversity at surrounding linked sites \cite{smith1974}.
To evaluate whether patterns of such ``hard sweeps'' could explain observed differences in diversity between genic and \DIFdelbegin \DIFdel{non-genic }\DIFdelend \DIFaddbegin \DIFadd{intergenic }\DIFaddend regions of the genome, we compared diversity around missense and synonymous substitutions between \emph{Tripsacum} and either maize or teosinte\DIFdelbegin \DIFdel{(Figure
\ref{fig:hardSweeps})}\DIFdelend .
If a proportion of missense mutations have been fixed due to hard sweeps, diversity around these substitutions should be lower than around synonymous substitutions. 
We observe this pattern around the causative amino acid substitution in the the domestication locus \emph{tga1} (Figure \ref{sFig:tga1}), likely the result of a hard sweep during domestication \cite{wang2005origin, wang2015}.
Genome-wide, however, we observe no differences in diversity between synonymous and missense substitutions in either maize or teosinte \DIFaddbegin \DIFadd{(Figure
\ref{fig:hardSweeps})}\DIFaddend .

\begin{figure*}
\centering
\includegraphics[width=.45\textwidth]{FigsAndFiles/plotDiversity_TvM_Folded2_Significance_Aug}
\hspace{0.05\textwidth} \includegraphics[width=.45\textwidth]{FigsAndFiles/plotDiversity_TvT_Folded2_Significance_Aug}
\caption{Pairwise diversity surrounding synonymous and \DIFdelbeginFL \DIFdelFL{non-synonymous (}\DIFdelendFL missense  \DIFdelbeginFL \DIFdelFL{)
  }\DIFdelendFL substitutions in {\bf A} maize and {\bf B} teosinte. Axes show \DIFdelbeginFL \DIFdelFL{both }\DIFdelendFL absolute diversity values (\DIFdelbeginFL \DIFdelFL{on }\DIFdelendFL right) and values relative to mean nucleotide diversity in windows $\geq 0.01 cM$ from a substitution (\DIFdelbeginFL \DIFdelFL{on }\DIFdelendFL left).  Lines depict a loess curve (span of 0.01) and shading represents bootstrap-based 95\% confidence intervals. Inset plots depict a larger range on the x-axis. \label{fig:hardSweeps}}
\end{figure*}

Previous analyses have suggested that this approach may have limited power because a \DIFdelbegin \DIFdel{higher proportion of nonsynonymous }\DIFdelend \DIFaddbegin \DIFadd{relatively high proportion of missense }\DIFaddend substitutions will be found in genes under weak purifying selection and thus \DIFdelbegin \DIFdel{with }\DIFdelend higher genetic diversity \cite{enard2014}. 
To address this concern, we took advantage of \DIFaddbegin \DIFadd{genome-wide estimates of evolutionary constraint \mbox{\cite{rodgers2015}}, calculated using }\DIFaddend genomic evolutionary rate profile (GERP) scores \cite{davydov2010}\DIFdelbegin \DIFdel{, a measure of evolutionary constraint, calculated across the maize genome \mbox{\cite{rodgers2015}}. 
We re-analyzed substitutions }\DIFdelend \DIFaddbegin \DIFadd{. 
We then evaluated substitutions only }\DIFaddend in subsets of genes \DIFdelbegin \DIFdel{with }\DIFdelend \DIFaddbegin \DIFadd{in }\DIFaddend the highest and lowest 10\% quantile of mean GERP score, putatively representing genes under the strongest and weakest purifying selection  (Figure \ref{sFig:consUncons}). 
As expected, we see higher diversity around substitutions in genes under weak purifying selection \DIFaddbegin \jri{there a table or figure we can reference?}\DIFaddend , but we still \DIFdelbegin \DIFdel{see }\DIFdelend \DIFaddbegin \DIFadd{find }\DIFaddend no difference between synonymous and missense substitutions in either subset of the data.
Taken together, these data suggest hard sweeps do not play a major role in patterning genic diversity in either maize or teosinte.

\subsection{Diversity is strongly influenced by purifying selection} 
Selection can also reduce diversity in functional regions of the genome via removal of deleterious mutations, a process known \DIFdelbegin \DIFdel{to }\DIFdelend as purifying or background selection \cite{charlesworth1993}.
We investigated purifying selection in maize and teosinte by evaluating the reduction of diversity \DIFdelbegin \DIFdel{within }\DIFdelend \DIFaddbegin \DIFadd{around }\DIFaddend genes.
Pairwise diversity is strongly reduced within genes for both maize and teosinte (Figure \ref{fig:purify}A) but recovers quickly at sites outside of genes, consistent with the low levels of linkage disequilibrium generally observed in maize \cite{tenaillon2002,chia2012}. 
The reduction in relative diversity is more pronounced in teosinte, \DIFdelbegin \DIFdel{however, }\DIFdelend reaching lower levels in genes and occurring over a wider region.  

Our initial comparison of synonymous and missense substitutions has low power to detect the effects of selection acting on multiple mutations or standing genetic variation, because in such cases diversity is not necessarily reduced \cite{innan2004,messer2013}. 
Such ``soft sweeps'' \DIFdelbegin \DIFdel{, however, }\DIFdelend are still expected to occur more frequently in functional regions of the genome and could provide an alternative explanation for the observed reduction of diversity in genes. 
To test this possibility, we performed a genome-wide scan for selection using \DIFdelbegin \DIFdel{a method }\DIFdelend \DIFaddbegin \DIFadd{the H12 statistic, }\DIFaddend expected to be \DIFdelbegin \DIFdel{reasonably }\DIFdelend sensitive to both hard and soft sweeps \cite{garud2015}. 
\DIFdelbegin \DIFdel{Even }\DIFdelend \DIFaddbegin \DIFadd{Qualitative differences between maize and teosinte remained unchanged even }\DIFaddend after removing genes in the top 20\% of \DIFdelbegin \DIFdel{the }\DIFdelend H12 \DIFdelbegin \DIFdel{statistic used to identify targets of selection, however, qualitative differences between maize and teosinte in patterns of diversity in and near genes remained unchanged }\DIFdelend (Figure \ref{sFig:H12}\DIFdelbegin \DIFdel{).
}
\DIFdelend \DIFaddbegin \DIFadd{A).
}\DIFaddend We interpret these combined results as suggesting that purifying selection has left a more pronounced signature in the teosinte genome due to the increased efficacy of selection resulting from differences in effective population size.

\begin{figure*}
\centering
\includegraphics[width=.45\textwidth]{FigsAndFiles/distanceToGene_WithSignificance_Folded2_manuscript.png} \includegraphics[width=.45\textwidth]{FigsAndFiles/distanceToGene_WithSignificance_Singletons_Downsampled_threeLines_manuscript.png}
\caption{Relative diversity versus distance to nearest gene in maize and teosinte. 
Shown are \textbf{A} pairwise nucleotide diversity and \textbf{B} singleton diversity.  
Relative diversity is calculated compared to the mean diversity in windows $\geq 0.01 cM$ or $\geq 0.02 cM$ from the nearest
gene for pairwise diversity and singletons, respectively. 
  Lines depict cubic smoothing splines with smoothing parameters chosen via generalized cross validation and shading depicts bootstrap-based 95\% confidence intervals.
  Inset plots depict a smaller range on the x-axis. \label{fig:purify}
  }
\end{figure*}

\subsection{Demography of maize domestication} To explore whether differences in the efficacy of purifying selection between maize and teosinte can be explained by demographic processes, we estimated the parameters of a simple domestication bottleneck model  (Figure \ref{fig:bottleneck}). 
The most likely model estimates an ancestral population mutation rate of $\theta=0.0147$ per bp, which translates to an effective population size of $N_a \approx 123,000$ individuals\DIFdelbegin \DIFdel{given the mutation rate \mbox{\cite{clark2005}}.
The maize population splits }\DIFdelend \DIFaddbegin \DIFadd{.
We estimate that the maize population split }\DIFaddend from teosinte $\approx 15,000$ generations in the past\DIFaddbegin \DIFadd{, }\DIFaddend with an initial size of only $\approx 5\% $ of \DIFaddbegin \DIFadd{the }\DIFaddend ancestral $N_a$. 
\DIFdelbegin \DIFdel{Following domestication we estimate considerable gene flow between the populations: $M_{tm} = N_a \times 1.1 \times 10^{-5}$  migrants per generation from teosinte to maize and $M_{mt} = N_a \times 1.4 \times 10^{-5}$ migrants from maize to teosinte. 
}\DIFdelend After its split from teosinte\DIFaddbegin \DIFadd{, }\DIFaddend our model posits exponential population growth in maize, estimating a final modern effective population size of $N_m \approx 370,000$.
\DIFaddbegin \DIFadd{Maize and teosinte continue to exchange migrants after the population split, with gene flow between the populations estimated at $M_{tm} =  1.1 \times 10^{-5} \times N_a $  migrants per generation from teosinte to maize and $M_{mt} =  1.4 \times 10^{-5} \times N_a$ migrants from maize to teosinte. 
}\DIFaddend 

\DIFdelbegin \DIFdelendFL \DIFaddbeginFL \begin{figure*}
\DIFaddendFL \centering
\includegraphics[width=.4\textwidth]{FigsAndFiles/DomesticationModel/domesticationModel.pdf}
\caption{Parameter estimates for a simple bottleneck model of maize domestication. See methods for details. \label{fig:bottleneck} }
\DIFdelbeginFL \DIFdelend \DIFaddbegin \end{figure*}
\DIFaddend 

\DIFdelbegin \DIFdel{Because our relatively small sample size limits our ability to characterize the rare variants that are most informative of population expansion, we }\DIFdelend \DIFaddbegin \DIFadd{In addition to our simple bottleneck model, we investigated two alternative approaches for demographic inference.  First, we }\DIFaddend took advantage of \DIFdelbegin \DIFdel{a complementary data set of }\DIFdelend \DIFaddbegin \DIFadd{genotyping data from }\DIFaddend more than 4,000 maize landraces \DIFdelbegin \DIFdel{collected from across the Americas }\DIFdelend \cite{Hearne2015} to estimate the modern maize effective population size \DIFdelbegin \DIFdel{from }\DIFdelend \DIFaddbegin \DIFadd{using }\DIFaddend low frequency variants  \DIFaddbegin \DIFadd{informative of population expansion}\DIFaddend . 
This analysis yields a much higher estimate of the modern maize effective population size at  $N_m \approx 993,000$.
\DIFdelbegin 
\DIFdelend Finally, we \DIFdelbegin \DIFdel{reconstructed the demographic history of maize and teosinte using a recently developed coalescent approach \mbox{\citex }that does not require conditioning on a particular model (Figure }\DIFdel{)}\DIFdelend \DIFaddbegin \DIFadd{applied a model-free coalescent approach \mbox{\cite{schiffels2014} }using a subset of our samples}\DIFaddend . 
Though this analysis \DIFdelbegin \DIFdel{does highlight a potential weakness with our assumption of a constant size for teosinte }\DIFdelend \DIFaddbegin \DIFadd{suggests non-equilibrium dynamics for teosinte not included in our initial model, it is nonetheless broadly consistent}\DIFaddend , \DIFdelbegin \DIFdel{it nonetheless is broadly consistent with our simple model,   }\DIFdelend identifying a clear domestication bottleneck followed by rapid population expansion \DIFdelbegin \DIFdel{(Figure }\DIFdelend \DIFaddbegin \DIFadd{in maize to an extremely large extant size of $\approx 10^9$ (Figure \ref{sFig:msmc}}\DIFaddend ). 

\subsection{Population expansion leads to stronger purifying selection in modern maize} Motivated by the rapid post-domestication expansion of maize evident \DIFdelbegin \DIFdel{from }\DIFdelend \DIFaddbegin \DIFadd{in }\DIFaddend our demographic analysis, we \DIFdelbegin \DIFdel{investigated whether patterns of diversity at low frequency }\DIFdelend \DIFaddbegin \DIFadd{reasoned that low-frequency }\DIFaddend --- and \DIFdelbegin \DIFdel{therefore on average }\DIFdelend \DIFaddbegin \DIFadd{thus }\DIFaddend younger --- polymorphisms \DIFdelbegin \DIFdel{would show similar results in comparisons of maize and teosinte.
We thus repeated our analysis of diversityusing alleles present in only a single individual in the sample. 
In direct contrast to patterns observed for pairwise nucleotide }\DIFdelend \DIFaddbegin \DIFadd{might show patterns distinct from pairwise diversity. 
Singleton diversity around missense and synonymous substitutions (Figure \ref{sFig:singleton}) appears nearly identical to results from pairwise }\DIFaddend diversity (Figure \DIFdelbegin \DIFdel{\ref{fig:purify}A), diversity of singletons was }\DIFdelend \DIFaddbegin \DIFadd{\ref{fig:hardSweeps}), providing little support for a substantial increase in the number or strength of hard sweeps occurring in maize. 
In contrast, we observe a qualitative shift in the effects of purifying selection: singleton polymorphisms are }\DIFaddend more strongly reduced in and near genes in maize than in teosinte (Figure \ref{fig:purify}B), even after downsampling our maize data to account for differences in sample size. \DIFaddbegin \DIFadd{This relationship again remained after removing the 20\% of genes with the highest H12 values (Figure \ref{sFig:H12})
}\DIFaddend Finally, while direct comparison of \DIFdelbegin \DIFdel{curves for }\DIFdelend pairwise and singleton diversity within taxa are consistent with at least some non-equilibrium dynamics in teosinte, as indicated by our coalescent-based demographic analysis, these too reveal much stronger differences in maize (Figure \ref{sFig:singletonPi}). 

\DIFdelbegin \DIFdel{These results suggest that demographic history and population size change have impacted the efficacy of purifying selection during maize evolution.
In contrast, analysis of singleton diversity around missense and synonymous substitutions (Figure \ref{sFig:singleton}) provides a nearly identical pattern to that shown in Figure \ref{fig:hardSweeps}, providing little support for a substantial increase in the number or strength of hard sweeps occurring in maize.  
}
\DIFdelend \section{Discussion}

\subsection{Hard sweeps do not shape genome-wide diversity in maize}

Our findings demonstrate that \DIFaddbegin \DIFadd{classic }\DIFaddend hard selective sweeps have not contributed substantially to genome-wide patterns of diversity in maize, \DIFdelbegin \DIFdel{and our analysis restricting to sites undergoing the strongest and weakest purifying selection confirm this result }\DIFdelend \DIFaddbegin \DIFadd{a result we show }\DIFaddend is robust to \DIFdelbegin \DIFdel{previously highlighted limitations \mbox{\cite{enard2014}}. 
Our approach does not address sweeps on noncoding variants, but it agrees with }\DIFdelend \DIFaddbegin \DIFadd{concerns about power due to the effects of background selection  \mbox{\cite{enard2014}}. 
Although our approach ignores the potential for hard sweeps in noncoding regions of the genome, }\DIFaddend a growing body of \DIFdelbegin \DIFdel{previous evidence arguing }\DIFdelend \DIFaddbegin \DIFadd{evidence argues }\DIFaddend against hard sweeps \DIFaddbegin \DIFadd{as the prevalent mode of selection }\DIFaddend shaping maize variability. 
\DIFdelbegin \DIFdel{For instance, population genetic studies of }\DIFdelend \DIFaddbegin \DIFadd{Among well-characterized domestication loci, only the gene }\emph{\DIFadd{tga1}} \DIFadd{shows evidence of a hard sweep on a missense mutation \mbox{\cite{wang2015}}, while several loci are consistent with ``soft sweeps''  from standing variation \mbox{\cite{studer2011,gallavotti2004role} }or multiple mutations \mbox{\cite{wills2013}}. 
Moreover, genome-wide studies of domestication \mbox{\cite{hufford2012}},  }\DIFaddend local adaptation \cite{Takuno15062015} and modern breeding \cite{beissinger2014} \DIFdelbegin \DIFdel{demonstrate }\DIFdelend \DIFaddbegin \DIFadd{all support }\DIFaddend the importance of \DIFdelbegin \DIFdel{selection on }\DIFdelend standing variation as \DIFdelbegin \DIFdel{opposed to on new mutations . Even during domestication, very few differences between }\DIFdelend \DIFaddbegin \DIFadd{the primary source of adaptive variation. 
Soft sweeps are expected to be common when $2N_e\mu_b \ge 1$, where $\mu_b$ is the mutation rate of beneficial alleles with selection coefficient $s_b$ \mbox{\cite{messer2013population}}.  
Assuming a mutation rate of $3 \times 10^{-8}$ \mbox{\citep{clark2005} }and that on the order of $\approx 1-5\%$ of mutations are beneficial \mbox{\citep{eyre2007distribution}}, this implies that soft sweeps should be common in both }\DIFaddend maize and teosinte \DIFdelbegin \DIFdel{were fixed \mbox{\cite{hufford2012}}. 
Moreover, of domestication loci that have been well studied from a population genetics standpoint, only one shows a signal of a hard sweep \mbox{\cite{wang2015}}, although there are several consistent with selection on standing variation\mbox{\cite{studer2011,wills2013} }or selection on multiple mutations \mbox{\cite{wills2013}}.  }\DIFdelend \DIFaddbegin \DIFadd{for mutational targets $>>10kb$ --- a plausible size for quantitative traits or for regulatory evolution targeting genes with large up- or down-stream control regions \mbox{\citep[e.g.][]{studer2011}}. 
Indeed, many adaptive traits in both maize \mbox{\citex }and teosinte \mbox{\citex }are highly quantitative }\jri{cite wallace for maize? weber and/or lauter for teosinte?} \DIFadd{and adaptation in both maize \mbox{\citep{hufford2012} }and teosinte \mbox{\citep{pyhajarvi2013complex} }has involved selection on regulatory variation.  }\jri{if number is nice, add something like "\X of the windows of top H12 score genome-wide occur outside of genes"}
\DIFaddend 

\DIFdelbegin \DIFdel{The lack of hard sweeps patterning diversity differs from drosophila \mbox{\cite{sattath2011} }and capsella }\DIFdelend \DIFaddbegin \jri{paragraph needs work. maybe eliminate/move? i do think we need some comparison to other stuff, but the broader Ne-selection efficiency argument should come later.}
\DIFadd{The absence of evidence for a genome-wide impact of hard sweeps differs markedly from observations in }\emph{\DIFadd{Drosophila}} \DIFadd{\mbox{\cite{sattath2011} }and }\emph{\DIFadd{Capsella}} \DIFaddend \cite{williamson2014}, but is consistent with \DIFdelbegin \DIFdel{humans \mbox{\cite{hernandez2011}}}\DIFdelend \DIFaddbegin \DIFadd{data from humans \mbox{\cite{hernandez2011,pritchard2010genetics}}}\DIFaddend . 
In a related study, \cite{corbett2015} showed that selection tends to have an elevated impact on reducing diversity in species with large population size compared to small. This may explain why maize and humans, both bottlenecked species, show similar patterns while drosophila and capsella \DIFdelbegin \DIFdel{are different. However, we also do not observe abundant hard sweeps in teosinte, which calls this hypothesis into question.
}\DIFdelend \DIFaddbegin \DIFadd{behave differently. \textcolor{red}{However, we also do not observe abundant hard sweeps in teosinte, which calls this explanation into question.}
}\DIFaddend 

\DIFdelbegin \DIFdel{Another possibility is that maize and teosinte don't show evidence of abundant hard sweeps because instead positive selection tends to operate in the form of soft sweeps. The theory of when soft sweeps are expected was worked out by \mbox{\cite{messer2013population}}. They showed that for $\theta_b = 2N\mu_b$, where $\mu_b$ is the mutation rate of beneficial alleles with selection coefficient $s_b > 0$, if $\theta_b > \frac{1}{log({Ns_b})}$, then soft sweeps in the form of adaptation from multiple mutations should be common. Conversely, for neutral sites that somehow become beneficial (e.g. change in environment), if $\theta_b > \frac{1}{log({2Ns_b})}$ then soft sweeps in the form of adaptation from standing neutral variation should be common. Alternatively, for previously deleterious sites that become beneficial, if $\theta > \frac{1}{log \left(1+\frac{2Ns_b}{2Ns_d + 1} \right)}$, where $s_d$ is the previous selection coefficient, soft sweeps again the form of adaptation from standing variation should be common. The implications of this theory are that soft sweeps are common when $\theta_b \geq 1$, which can be achieved through a large population size or a high mutation rate. If $\mu = 3 \times 10^{-8}$  \mbox{\cite{clark2005}}, this implies a population size of $\geq$ 16.67 million, which our demographic analysis shows is larger than that of teosinte but not out of the question for modern maize.
}
\DIFdelend \subsection{Demography of domestication}
\DIFdelbegin 
\DIFdel{Although many other }\DIFdelend \DIFaddbegin \DIFadd{Although a number of }\DIFaddend authors have investigated the demography of maize domestication \cite{eyre1998, tenaillon2004selection, wright2005}, these  efforts relied \DIFdelbegin \DIFdel{exclusively genic polymoprhisms and have }\DIFdelend \DIFaddbegin \DIFadd{on data only from genic regions of the genome and }\DIFaddend made a number of limiting assumptions about the demographic model.  
We show that diversity within genes has been strongly reduced by the effects of linked selection, \DIFdelbegin \DIFdel{making genic data inappropriate for estimating demography.
Because levels of diversity alone do not allow independent inference of the strength and duration of a bottleneck \mbox{\cite{tenaillon2004selection}}, previous investigations have instead estimated the ratio of these parameters.  
We have overcome this limitations through the use of the }\DIFdelend \DIFaddbegin \DIFadd{such that even synonymous polymorphisms in genes are not representative of diversity at unconstrained sites.  
By utilizing the }\DIFaddend full joint SFS\DIFdelbegin \DIFdel{of maize and teosinte and consideration of a more realistic model of exponential growth, enabling us to estimate for the first time the }\DIFdelend \DIFaddbegin \DIFadd{, we are able to estimate population growth, gene flow, and the }\DIFaddend strength of the domestication bottleneck \DIFdelbegin \DIFdel{.  
Given that modern maize exhibits $\approx 80\%$ of the diversity of teosinte, our estimate that  the effective population size of the initial maize  population was only $\approx 5\%$ of the ancestral teosinte population seems surprisingly low.
The impact of a strong domestication bottleneck is likely ameliorated to some degree by  maize-teosinte gene flow and the rapid post-domestication growth estimated in our model.   
}\DIFdelend \DIFaddbegin \DIFadd{without requiring assumptions about its duration.  
}\DIFaddend 

\DIFdelbegin \DIFdel{Another }\DIFdelend \DIFaddbegin \DIFadd{One }\DIFaddend surprising result from our model is the estimation of the timing of domestication \DIFdelbegin \DIFdel{.
Although we estimate the maize population split $\approx 5,000$ years earlier than previous molecular \mbox{\cite{matsuoka2002} }or archaelogical }\DIFdelend \DIFaddbegin \DIFadd{at $\approx 15,000$ years before present.
While this appears to conflict with archaeological }\DIFaddend \cite{piperno2009starch} estimates, we \DIFdelbegin \DIFdel{do not interpret these results as strongly conflicting.
We first note that the timing of the }\DIFdelend \DIFaddbegin \DIFadd{instead argue this estimate reflects the fact that the }\DIFaddend genetic split between populations likely preceded \DIFdelbegin \DIFdel{the }\DIFdelend anatomical changes that can be identified in the archaeological record. 
We also \DIFdelbegin \DIFdel{caution, however, that our estimate may be inflated by population structure}\DIFdelend \DIFaddbegin \DIFadd{note that our result may also be inflated due to population structure, }\DIFaddend as our geographically diverse sample \DIFaddbegin \DIFadd{of teosinte }\DIFaddend may include populations diverged from those that gave rise to maize.

\DIFaddbegin \DIFadd{The estimated bottleneck of $\approx 5\%$ of the ancestral teosinte population seems low given that maize landraces exhibit $\approx 80\%$ of the diversity of teosinte \mbox{\citep{hufford2012}}, but our model suggests that the effects of the bottleneck on diversity are likely ameliorated to some extent by both gene flow and rapid post-domestication growth (Figure \ref{fig:bottleneck}).   
}\DIFaddend Although we estimate that the modern effective size of maize is \DIFdelbegin \DIFdel{three times that of }\DIFdelend \DIFaddbegin \DIFadd{larger than }\DIFaddend teosinte, the small size of our sample \DIFdelbegin \DIFdel{dramatically }\DIFdelend reduces our power to \DIFdelbegin \DIFdel{estimate }\DIFdelend \DIFaddbegin \DIFadd{identify the low frequency alleles most sensitive to }\DIFaddend rapid population growth \DIFdelbegin \DIFdel{since rapid growth is most evidenced by rare alleles}\DIFdelend \cite{keinan2012}.  
A better estimate \DIFdelbegin \DIFdel{of the effective size of modern maize }\DIFdelend comes from our analysis of singleton diversity in a large sample of maize landraces \cite{Hearne2015}\DIFdelbegin \DIFdel{.
These data suggest }\DIFdelend \DIFaddbegin \DIFadd{, which suggests }\DIFaddend a modern effective size of \DIFaddbegin \DIFadd{$\approx 1$ million, or }\DIFaddend nearly \DIFdelbegin \DIFdel{1 million, nearly }\DIFdelend eight times higher than modern teosinte.
\DIFdelbegin \DIFdel{Nonetheless, }\DIFdelend \DIFaddbegin \DIFadd{Even this estimate, however, is likely downwardly biased due to }\DIFaddend missing data, sample size, and ascertainment \DIFdelbegin \DIFdel{bias }\DIFdelend in the genotyping data\DIFdelbegin \DIFdel{all bias this estimate downward as well.
We can, however, get reasonable }\DIFdelend \DIFaddbegin \DIFadd{.
An alternative approach is to use }\DIFaddend estimates of the census size of modern maize\DIFdelbegin \DIFdel{.
There }\DIFdelend \DIFaddbegin \DIFadd{: there }\DIFaddend are 47.9 million ha of open-pollinated maize in production \cite{cimmyt1999}, \DIFdelbegin \DIFdel{with a lower limit of 25,000 individuals  planter }\DIFdelend \DIFaddbegin \DIFadd{likely planted at a density of $\approx 25,000$ individuals  }\DIFaddend per hectare \cite{baden2001culture}.
\DIFdelbegin \DIFdel{Using this lower bound on planting density, ignoring all industrial maize agriculture, and even assuming }\DIFdelend \DIFaddbegin \DIFadd{Assuming }\DIFaddend the effective size is only \DIFdelbegin \DIFdel{0.1\% }\DIFdelend \DIFaddbegin \DIFadd{$\approx 0.4\%$ }\DIFaddend of the census size \DIFaddbegin \DIFadd{(i.e. 1 ear for every 1000 plants)}\DIFaddend , this still implies a modern effective population size of more than \DIFdelbegin \DIFdel{one billion.
Clearly, the }\DIFdelend \DIFaddbegin \DIFadd{four billion.
While these rough estimates are likely poor, all of the evidence points to the fact that the }\DIFaddend effective size of modern maize is extremely large.

\subsection{Demography influences the efficiency of purifying selection}
\DIFdelbegin \DIFdel{The observation that maize pairwise diversity }\DIFdelend \DIFaddbegin \jri{section needs help --  i'm not happy with the writing} 
\DIFadd{We find a more pronounced decrease in nucleotide diversity around genes in teosinte compared to domesticated maize (Figure \ref{fig:purify}A).
This difference is likely due to the effects of purifying selection.
Purifying selection }\DIFaddend is \DIFdelbegin \DIFdel{less impacted by the distance to the nearest gene than is teosinte pairwise diversity is reasonable from a long-term evolutionary standpoint. Theory has established that purifying selectionis }\DIFdelend more efficient in \DIFdelbegin \DIFdel{a large population than in a small one }\DIFdelend \DIFaddbegin \DIFadd{larger populations  }\DIFaddend \cite{kimura1984}, and \DIFdelbegin \DIFdel{this observation most likely reflects that prediction. More specifically, if teosinte $N_e$ remained relatively constant while maize bottlenecked and recovered exponentially, this provides that the average $N_e$ of maize over the previous several thousand generations is much smaller than that of teosinte, regardless of how much maize has ballooned in the recent past. Therefore, our observation shows that purifying selection in maizehas not purged deleterious alleles, or the neutral alleles they are linked to, as effectively as in teosinte. }
\DIFdel{The reversal of this trend when we analyze only singleton diversity instead of pairwise diversity, however, stands out as a notable observation.   Every mutation begins as a singleton (an allele present in only one individual), and therefore singletons are , on average, the youngest class of alleles that can be observed. Therefore unlike pairwise }\DIFdelend \DIFaddbegin \DIFadd{our observation is thus consistent with our demographic model and previous work \mbox{\cite{eyre1998, tenaillon2004selection, wright2005, ross2009} }showing a smaller long-term effective population size in maize.  
To understand how recent population growth has changed the effects of linked selection, we also analyzed diversity of singleton polymorphisms.   Under a simple neutral model, lower frequency polymorphisms are expected to be younger, and }\DIFaddend patterns of diversity \DIFdelbegin \DIFdel{, which are most heavily influenced by intermediate frequency alleles based on the definition of $\pi$ \mbox{\cite{nei1979}}, singleton diversity is most influenced by recent patterns of evolution. 
Hence, because our demographic estimation indicates dramatic expansion of maize $N_e$ in the recent past, we expect for purifying selectionto presently operate more efficiently in maize than in teosinte. This observation is very clearly demonstrated by the fact that singleton diversity in maize is more impacted by the distance to the nearest gene than it is in teosinte, as was shown in Figure \ref{fig:purify}. }\DIFdelend \DIFaddbegin \DIFadd{at singleton SNPs should thus represent recent evolutionary history. 
Consistent with its larger current size and thus more efficient purifying selection, we see a stronger decline of singleton diversity around genes in maize (Figure \ref{fig:purify}B than teosinte.
}\DIFaddend 

\DIFaddbegin \jri{I would like to see a supplemental figure or maybe just number here on the partial correlation in both teosinte and maize of diversity with recombination after accounting for gene density.  do this in 500kb windows and then in 10kb windows.  we should do the correlation with both pi and singletons.  This let's us compare more explictly with studies like C-D who evaluate overall patterns of linked selection. will be especially cool if we don't see a big difference in big windows but we do in 10kb windows.  }
\DIFaddend A consequence of the inefficient purifying selection that maize experienced during its bottleneck is likely that it harbors more \DIFaddbegin \DIFadd{weakly }\DIFaddend deleterious alleles segregating at intermediate frequency than does teosinte. This could be a part of the explanation of why maize inbreds have continued to improve over the past several decades \cite{meghji1984}; if deleterious alleles tend to be recessive and are particularly frequent, they will have ample opportunities to display their phenotypes in inbreds. Our results also demonstrate that recent purifying selection in maize has become much more effective, potentially explaining the ongoing improvement of these inbreds as maize lines are continuously selected. \DIFdelbegin \DIFdelend \DIFaddbegin \DIFadd{Additionally, the large $N_e$ of modern maize compared to modern teosinte implies that for new mutations, selection will operate much more efficiently in maize. }\jri{inbreds stuff OK, but should be shortened.  and we need more/broader. Chia, Rogers-Melnick, Mezmouk all looked at load. does this have implications for large effect load (i.e. should bneck have purged those?), for heterosis, architecture of quant. traits, etc.}
\DIFaddend 

\DIFdelbegin \DIFdel{Importantly, our estimation of the parameters of the maize domestication bottleneck contribute to the understanding of how the demography of crop domestication can impact crop diversity. The bottleneck-effects from a sudden collapse in population size have been well studied and are known to impact crops for thousands of generations. Complementing this knowledge, our results demonstrate that the rapid expansion experienced by many crops after domestication can also have a profound influence on patterns of diversity, and the effects of this expansion should be accounted for as important contributors to long-term evolution. }\DIFdelend \DIFaddbegin \jri{these results have implications for lots of studies. recent demographic history often ignored (examples). in crops, bneck is considered and loss of diversity, but expansion also likely ubiquitous and consequences not well studied.  in humans XYZ, and we see some of that already here. this argues that a full understanding of diversity in both wild and cultivated plants will require recent demography. mention C-D ignored recent demography -- here's where rec/pi windows could help.}
\DIFaddend 

\DIFdelbegin 
\DIFdelend \begin{materials}

\subsection{BASH, R, and Python scripts}
All scripts used for analysis are available in an online repository at \textcolor{red}{REPO ADDRESS HERE.} 

\subsection{Plant materials}
We made use of published sequences from inbred accessions of teosinte (Z. \emph{mays} ssp. \emph{parviglumis}) and maize landraces from the Maize HapMap3 panel as part of the Panzea project (bam files are available at \textcolor{red}{/iplant/home/shared/panzea/hapmap3/bam\_internal/v3\_bams\_bwamem}) \cite{chia2012, lemmon2014,hapmap3}. 
From these data, we removed 4 teosinte individuals that were not ssp. \emph{parviglumis} or appeared as outliers in an initial principal component analysis conducted with the package adegenet \cite{jombart2011} (Figure \ref{sFig:PCA}), leaving 13 teosinte and 23 maize that were used for all subsequent analyses (Table \ref{sTab:list}). We also utilized a single tripsacum (\emph{T. dactyloides}) individual as an outgroup (bam file available at \textcolor{red}{/iplant/home/shared/panzea/hapmap3/bam\_internal/v3\_bams\_bwamem}). \DIFaddbegin \jri{some clarification here. did we use Lemmon's higher depth teosinte? If so, why does Li say we only had 7 high-depth bam files? there should be more}
\DIFaddend 

\subsection{Physical and genetic maps}
Sequences were mapped to the maize B73 version 3 reference genome \cite{schnable2009} (ftp://ftp.ensemblgenomes.org/pub/plants/release-22/fasta/zea\_mays/dna/) as described by \cite{hapmap3}. All analyses made use of uniquely mapping reads with mapping quality score $\geq  30$ and bases with base quality score $\geq 20$; quality scores around indels were adjusted following Li \emph{et al.} \cite{li2011statistical}.
We converted physical coordinates to genetic coordinates via linear interpolation of the previously published 1cM resolution NAM genetic map \cite{glaubitz2014}. 

\subsection{Estimating the site frequency spectrum}
We estimated both the genome-wide site frequency spectrum (SFS) as well as a separate SFS for genic (within annotated transcript) and intergenic ($\geq 5kb$ from a transcript) regions. 
We used the biomaRt package \cite{durinck2009,durinck2005} of R \cite{R2014} to parse annotations from genebuild version 5b of AGPv3. 
We estimated single population and joint SFS with the software ANGSD \cite{korneliussen2014}, including all positions with at least one aligned read in $\geq 80\%$ of samples in one or both populations.
We assumed individuals were fully inbred and treated each line as a single haplotype. Because ANGSD cannot calculate a folded joint SFS, we first polarized SNPs using the maize reference genome and then folded spectra using $\delta\alpha\delta{i}$ \cite{gutenkunst2009}.

\subsection{Demographic inference}
We used the software $\delta\alpha\delta{i}$ \cite{gutenkunst2009} to estimate parameters of a domestication bottleneck from the joint maize-teosinte SFS, using only sites $>5 kb$ from a gene to ameliorate the effects of linked selection.
We modeled a teosinte population of constant effective size $N_a$, that at time $T_b$ generations in the past gave rise to a maize population of size $N_b$ which grew exponentially to size $N_m$ in the present (Figure \ref{fig:bottleneck}).
The model includes migration of $M_{mt}$ individuals each generation from maize to teosinte and $M_{tm}$ individuals from teosinte to maize.  We estimated $N_a$ using $\delta\alpha\delta{i}$'s estimation of $\theta=4N_a\mu$ from the data and a mutation rate of $\mu = 3 \times 10^{-8}$ \cite{clark2005}. 
We estimated all other parameters using 1,000 $\delta\alpha\delta{i}$ optimizations and allowing initial values between runs to be randomly perturbed by a factor of 2.  
Optimized parameters along with their initial values and upper and lower bounds can be found in table \ref{sTab:dadi}. We report parameter estimates from the optimization run with the highest log-likelihood.

We further made use of a large genotyping data set of more than 4,000 maize landraces \cite{Hearne2015} to estimate the modern maize $N_e$ from singleton counts.
We filtered these data to include only SNPs with data in $\geq 1,500$ individuals, and then projected the SFS down to a sample of 500 individuals by sampling each marker without replacement 1,000 times according to the observed allele frequencies.
We then estimated $N_e$ from the data assuming $\mu = 3 \times 10^{-8}$ \cite{clark2005} and the relation  $4N_e\mu = \frac{S}{L}$ \cite{fu1993}, where where $S$ is the total number of singleton SNPs and $L$ is the total number of SNPs in the dataset.

\DIFdelbegin \DIFdel{Finally, }\DIFdelend \DIFaddbegin \DIFadd{As a final estimate of demography, we employed MSMC \mbox{\cite{schiffels2014}  }to complement our model-based demographic inference. 
We used six each of maize and teosinte (BKN022, BKN025, BKN029, BKN030, BKN031, BKN033, TIL01, TIL03, TIL09, TIL10, TIL11 and TIL14), treating each inbred genome as a single haplotype.
We called SNPs in ANGSD \mbox{\cite{korneliussen2014} }using a SNP p-value of $1e-6$ against a reference genome masked using SNPable (}\url{http://lh3lh3.users.sourceforge.net/snpable.shtml}\DIFadd{).
We then removed heterozygous genotypes and filtered sites with a mapping quality $<30$, a base quality $<20$, or a $|\emph{log}_2(\emph{depth})|<1$.
We ran MSMC with pattern parameters $20\times2+20\times4+10\times2$.
}\DIFaddend 

\subsection{Diversity}
We made use of the software ANGSD \cite{korneliussen2014} for diversity calculations and genotype calling. 
We calculated diversity statistics in maize and teosinte in 1 kb non-overlapping windows using filters as described above for the SFS. 
We used allele counts to estimate the number of singleton polymorphisms in each window, and used binomial sampling to create a second maize data set down-sampled to have the same number of samples as teosinte.
We called genotypes in maize, teosinte, and \emph{Tripsacum} at sites with a SNP p-value $<10^{-6}$ and when the genotype posterior probability $>0.95$.
We identified substitutions in maize and teosinte as all sites with a fixed difference with \emph{Tripsacum} and $\leq 20\%$ missing data. 
Substitutions were classified as synonymous, missense, or noncoding using the ensembl variant effects predictor \cite{mclaren2010}.
For each window with $\geq 100bp$ of data we computed the genetic distance between the window center and the nearest synonymous and missense substitution as well as the genetic distance to the center of the nearest gene transcript.  

\subsection{Selection scan}
We scanned the genome to identify sites that have experienced recent positive selection using the H12 statistic \cite{garud2015} in sliding windows of 200 SNPs with a step of 25 SNPs.

\end{materials}

\begin{acknowledgments}
We are indebted to Graham Coop and Simon Aeshbacher for their constructive input during this study. \DIFdelbegin \DIFdel{\textcolor{red}{We thank Robert Bukowski and Qi Sun for providing early-access Panzea data.} }\DIFdelend \DIFaddbegin \DIFadd{We thank Robert Bukowski and Qi Sun for providing early-access data from maize HapMap3. }\DIFaddend Funding was provided by NSF Plant Genome Research Project 1238014.
\end{acknowledgments}

\bibliography{Reference}
\bibliographystyle{pnas}
\onecolumn
\section*{Supporting Information}
\renewcommand\thefigure{S\arabic{figure}}    
\setcounter{figure}{0}


\begin{figure}
  \begin{center}
    \includegraphics[width=.85\textwidth]{FigsAndFiles/plotDiversity_TvM_Folded2_Significance_tga1Supp_June.png} \\
    \end{center}
\caption{Diversity surrounding the causitive polymorphism at the \emph{tga1} locus is plotted. Since this is only one gene, the large amount of noise compared to our average plots is expected. However, notice that diversity precisely at the causitive polymorphism is reduced and a recovery of diversity is observed away from that site. \label{sFig:tga1}}
\end{figure}
\clearpage


\begin{figure}
  \DIFaddbeginFL \begin{center}
    \includegraphics[width=.85\textwidth]{FigsAndFiles/BKN.TIL.msmc.pdf} \\
    \end{center}
\caption{\jri{need MSMC caption} \label{sFig:msmc}}
\end{figure}
\clearpage


\begin{figure}
  \DIFaddendFL \includegraphics[width=.5\textwidth]{FigsAndFiles/plotDiversity_TvM_Conserved_Significance_June.png}
  \includegraphics[width=.5\textwidth]{FigsAndFiles/plotDiversity_TvM_Unconserved_Significance_June.png}
\caption{ Pairwise diversity surrounding synonymous and nonsynonymous
  substitutions in maize at highly conserved (A) or unconserved (B) sites.  Bootstrap-based 95\% confidence intervals are depicted via shading. Inset plots depict a larger range on the x-axis. \label{sFig:consUncons}}
\end{figure}
\clearpage

\begin{figure}
  \includegraphics[width=\textwidth]{FigsAndFiles/plotDiversity_TvM_Singletons.png}
\caption{ Singleton diversity surrounding synonymous and nonsynonymous
  substitutions in maize. \label{sFig:singleton}}
\end{figure}
\clearpage


\begin{figure}
  \includegraphics[width=\textwidth]{FigsAndFiles/distanceToGene_WithSignificance_Folded2_maizeAndTeoSingleVsPi.png}
\caption{ Relative diversity versus distance to nearest gene in maize and teosinte. Relative diversity is calculated by comparing to the mean diversity in all windows $\geq 0.02 cM$ from the nearest gene. Lines depict cubic smoothing splines with smoothing parameters chosen via generalized cross validation and shading depicts bootstrap-based 95\% confidence intervals.  \label{sFig:singletonPi}}
\end{figure}
\clearpage


\begin{figure}
  \includegraphics[width=.5\textwidth]{FigsAndFiles/distanceToGene_Unselected_manuscript.png}
    \includegraphics[width=.5\textwidth]{FigsAndFiles/distanceToGene_unselected_Singletons_manuscript.png}
\caption{ Relative level of diversity versus distance to the nearest gene, in maize and teosinte, based on only sites that do not show evidence of hard or soft sweeps according to H12. Two measures of diversity were investigated. {\bf A} displays pairwise diversity,
which is most influenced by intermediate frequency alleles and therefore depicts more ancient evolutionary patterns, and {\bf B} depicts singleton diversity, influenced by rare alleles and thus depicting evolutionary patterns in the recent past. Bootstrap-based 95\% confidence intervals are depicted via shading. Inset plots depict a smaller range on the x-axis. \label{sFig:H12}}
\end{figure}
\clearpage


\begin{figure}
  \begin{center}
  \includegraphics[width=.75\textwidth]{FigsAndFiles/tilPCA_aug.pdf}\\
  \includegraphics[width=.75\textwidth]{FigsAndFiles/bknPCA_aug.pdf}\\
  \end{center}
  \caption{Principal component analysis of teosinte and maize individuals to ensure that no close relatives were inadvertantly included in our study. Plots are based on a random sample of 10,000 SNPs. {\bf A:} Percentage of total variance explained by each principal component for teosinte. {\bf B:} PC1 vs PC2 for all 13 teosinte individuals. {\bf C:} Percentage of total variance explained by each principal component for maize. {\bf D:} PC1 vs PC2 for all 23 maize individuals. \label{sFig:PCA}}
\end{figure}
\clearpage


\begin{figure}
  \begin{tabular}{c|c}
    \bf Maize & \bf Teosinte \\ \hline \hline
    BKN009 &  TIL01 \\
    BKN010 & TIL02 \\
    BKN011 & TIL03 \\
    BKN014 & TIL04-TIP454 \\
    BKN015 & TIL07 \\
    BKN016 & TIL09 \\
    BKN017 & TIL10 \\
    BKN018 & TIL11 \\
    BKN019 & TIL12 \\
    BKN020 & TIL14-TIP498 \\
    BKN022 & TIL15 \\
    BKN023 & TIL16 \\
    BKN025 & TIL17 \\
    BKN026 & \\
    BKN027 & \\
    BKN029 & \\
    BKN030 & \\
    BKN031 & \\
    BKN032 & \\
    BKN033 & \\
    BKN034 & \\
    BKN035 & \\
    BKN040 & \\
  \end{tabular} 
  \caption{ A list of maize and teosinte individuals included in this study. Sequencing and details were previously described by \jri{cite chia and lemmon}   \label{sTab:list} }
\end{figure}
\clearpage

\begin{figure}
  \def\arraystretch{2}
  \begin{tabular}{l|c|c|c}
    \bf Parameter & \bf Initial value & \bf Upper bound & \bf Lower bound\\ \hline 
    $\frac{N_b}{N_a}$ & 0.02 & $1\times10^{-7}$ & 2 \\ 
    $\frac{N_{m}}{N_a}$ & 3 & $1\times10^{-7}$ & 200 \\
    $\frac{T_b}{2N_a}$ & 0.04 & 0  & 1 \\ 
    $\frac{M_{mt}}{N_a}$ & $1\times10^{-10}$ & $1\times10^{-7}$ & 0.001 \\
    $\frac{M_{tm}}{N_a}$ & $1\times10^{-10}$ & $1\times10^{-7}$ & 0.001 \\
  \end{tabular} 
  \caption{ Parameters, initial values, and boundaries used for model-fitting with $\delta\alpha\delta{i}$. Parameters are shown in the units utilized by $\delta\alpha\delta{i}$, although in the text simplified units are reported.   \label{sTab:dadi} }
    \def\arraystretch{1} \end{figure}
\clearpage



\DIFdelbegin 
\DIFdelend \end{article}
\DIFaddbegin 


 \DIFaddend\end{document}
