\documentclass[12pt,a4paper]{article}
\usepackage{cite}
\usepackage{color}
\usepackage{pgf}
\usepackage[margin=1in]{geometry}
%\usepackage{hyperref}
\usepackage[normalem]{ulem}
\newcommand{\jri}[1]{\textcolor{red}{\scriptsize #1}}
\newcommand{\mbh}[1]{\textcolor{blue}{\scriptsize #1}}
\newcommand{\tmb}[1]{\textcolor{brown}{\scriptsize #1}}
\newcommand{\citex}{\textcolor{red}{\bf CITE}}
\newcommand{\X}{\textcolor{red}{\bf X}}
\usepackage{natbib}
\usepackage{authblk}
\usepackage[colorlinks=true,
            linkcolor=red,
            urlcolor=blue,
            citecolor=gray]{hyperref}
\usepackage{natbib}
\bibpunct{}{}{,}{s}{}{\textsuperscript{,}}
\usepackage{amsmath}
\usepackage{graphicx}

%\renewcommand{\abstractname}{\vspace{-\baselineskip}}
\renewcommand{\abstractname}{Abstract (opening paragraph)}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\title{More than one Author with different Affiliations}
\author[1,2,3,*]{Timothy M. Beissinger}
\author[4]{Li Wang}
\author[1]{Kate Crosby}
\author[1]{Arun Durvasula}
\author[4]{Matthew B. Hufford}
\author[1,5,*]{Jeffrey Ross-Ibarra}
\affil[1]{Dept. of Plant Sciences, University of California, Davis, CA, USA}
\affil[2]{US Department of Agriculture, Agricultural Research Service, Columbia, MO, USA}
\affil[3]{Division of Plant Sciences, University of Missouri, Columbia, MO, USA}
\affil[4]{Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, IA, USA}
\affil[5]{Genome Center and Center for Population Biology, University of California, Davis, CA, USA}
\affil[*]{Correspondence may be addressed to rossibarra@ucdavis.edu or beissingert@missouri.edu}

\renewcommand\Authands{ and }


\newcounter{firstbib} % bibliography counter

\begin{document}

\title{Recent demography drives changes in linked selection across the maize genome}

\maketitle
%\cofeAm{1}{1.0}{0}{5.5cm}{3cm}

\begin{abstract}
Genetic diversity is shaped by the interaction of drift and selection, but the details of this interaction are not well understood. The impact of genetic drift in a population is largely determined by its demographic history, typically summarized by its long-term effective population size ($N_e$ ).  Rapidly changing population demographics complicate this relationship, however. To better understand how changing demography impacts selection, we used whole-genome sequencing data to investigate patterns of linked selection in domesticated and wild maize (teosinte). We produce the first whole-genome estimate of the demography of maize domestication, showing that maize was reduced to approximately 5\% the population size of teosinte before it experienced rapid expansion post-domestication to population sizes much larger than its ancestor. Evaluation of patterns of nucleotide diversity in and near genes shows little evidence of selection on beneficial amino acid substitutions, and that the domestication bottleneck led to a decline in the efficiency of purifying selection in maize.  Young alleles, however, show evidence of much stronger purifying selection in maize, reflecting the much larger effective size of present day populations. Our results demonstrate that recent demographic change --- a hallmark of many species including both humans and crops ---  can have immediate and wide-ranging impacts on diversity that conflict with would-be expectations based on $N_e$ alone.
\end{abstract}

The genetic diversity of populations is determined by a constant interplay between genetic drift and natural selection. 
Drift is a consequence of a finite population size and the random sampling of gametes each generation \cite{dobzhansky1957}. 
In contrast to the stochastic effects of drift, selection systematically alters allele frequencies by favoring particular alleles at the expense of others as a result of their effects on fitness.
Researchers often study drift by excluding potentially selected sites \cite{voight2005, gutenkunst2009}, or selection by focusing on site-specific patterns under the assumption that genome-wide diversity reflects primarily the action of drift \cite{akey2009}.

Drift and selection do not operate independently to determine genetic variability, however, in large part because linkage allows the effects of selection to be wide-ranging \cite{smith1974, slotte2014}. 
Linked selection, which refers to the effects of selection at one site on diversity at linked sites \cite{slotte2014}, can take the form of hitch-hiking, when the frequency of a neutral allele changes as a result of positive selection at a physically linked site \cite{smith1974}, or background selection, where diversity is reduced at loci linked to a site undergoing selection against deleterious alleles \cite{charlesworth1993}. 
Recent work in \emph{Drosophila}, for example, has shown that virtually the entire genome is impacted by the combined effects of these processes \cite{sella2009,elyashiv2014,andolfatto2005}.

The impact of linked selection, in turn, is heavily influenced by the effective population size ($N_e$), as the  efficiency of natural selection is proportional to the product $N_es$, where $s$ is the strength of selection on a variant \cite{cutter2013, slotte2014, corbett2015,leffler2012}.
The effective size of a population is not static, and nearly all species, including flies \cite{duchen2013}, humans \cite{reich1998}, domesticates \cite{meyer2013}, and non-model species \cite{ellegren2014} have experienced recent or ancient changes in $N_e$. 
Although much is known about how the long-term average $N_e$ affects linked selection \cite{cutter2013}, relatively little is understood about the immediate effects of more recent changes in $N_e$ on patterns of linked selection.

Because of its relatively simple demographic history and well-developed genomic resources, maize (\emph{Zea mays}) represents an excellent organism to study these effects. 
Archaeological and genetic studies have established that maize domestication began in Central Mexico at least 9,000 years bp \cite{matsuoka2002}, and involved a population bottleneck followed by recent expansion \cite{wright2005,eyre1998,tenaillon2004}.
Because of this simple but dynamic demographic history, domesticated maize and its wild ancestor teosinte can be used to understand the effects of changing $N_e$ on linked selection. 
In this study, we leverage the maize-teosinte system to study these effects by first estimating the parameters of the maize domestication bottleneck using whole-genome resequencing data and then investigating the relative importance of different forms of linked selection on diversity in the ancient and more recent past. 
We show that, while patterns of overall nucleotide diversity reflect long-term differences in $Ne$, recent growth following domestication qualitatively changes these effects, thereby illustrating the importance of a comprehensive understanding of demography when considering the effects of selection genome-wide. 

\section*{RESULTS}

\subsection*{Patterns of diversity differ between genic and  intergenic regions of the genome} %JRI
To investigate how demography and linked selection have shaped patterns of diversity in maize and teosinte, we analyzed data from 23 maize and 13 teosinte genomes from the maize HapMap 2 and HapMap 3 projects \cite{chia2012, bukowski2015}. As a preliminary step, we evaluated levels of diversity inside and outside of genes across the genome. We find broad differences in genic and intergenic diversity consistent with earlier results  \cite{hufford2012}(Figure \ref{fig:diversity}).  In maize, mean pairwise diversity ($\pi$) within genes was significantly lower than at sites at least 5 kb away from genes (0.00668 vs 0.00691, $p<2\times 10^{-44}$). 
Diversity differences in teosinte are even more pronounced (0.0088 vs. 0.0115, $p\approx 0$). 
Differences were also apparent in the site frequency spectrum, with mean Tajima's D positive in genic regions in both maize (0.4) and teosinte (0.013) but negative outside of genes (-0.087 in maize and -0.25 in teosinte, $p\approx 0$ for both comparisons).
These observations suggest that diversity in genes is not evolving neutrally, but instead is reduced by the impacts of selection on linked sites. 



\subsection*{Demography of maize domestication} %JRI
We next estimated a demographic model of maize domestication (Figure \ref{fig:bottleneck}). 
To minimize the impact of selection on our estimates \cite{ewing2016}, we only included sites \textgreater 5kb from genes.  
The most likely model estimates an ancestral population mutation rate of $\theta=0.0147$ per bp, which translates to an effective population size of $N_a \approx 123,000$ teosinte individuals.
We estimate that maize split from teosinte $\approx 15,000$ generations in the past, with an initial size of only $\approx 5\% $ of the ancestral $N_a$. 
After its split from teosinte, our model posits exponential population growth in maize, estimating a final modern effective population size of $N_m \approx 370,000$.
Although our model provides only a rough approximation of migration rates, we included migration parameters during demographic inference because omitting these could bias our population size estimates. 
We observe that maize and teosinte have continued to exchange migrants after the population split, with gene flow from teosinte to maize was $M_{tm} =  1.1 \times 10^{-5} \times N_a $  migrants per generation, and from maize to teosinte we estimate $M_{mt} =  1.4 \times 10^{-5} \times N_a$ migrants per generation. 

Because our modest sample size of fully sequenced individuals has limited power to infer recent population expansion, we investigated two alternative approaches for demographic inference.  
First, we utilized genotyping data from more than 4,000 maize landraces \cite{Hearne2015} to estimate the modern maize effective population size.
Because rare variants provide the best information about recent effective population sizes \cite{keinan2012}, we estimate $N_e$ using  a singleton-based estimator \cite{fu1993} of the population mutation rate $\theta=4N_e\mu$  and published values of the mutation rate \cite{clark2005} (see online methods for details). 
This yields a much higher estimate of the modern maize effective population size at  $N_m \approx 993,000$.
Finally, we employed a model-free coalescent approach \cite{schiffels2014} to estimate population size change using a subset of six genomes each of maize and teosinte.
Though this analysis suggests non-equilibrium dynamics for teosinte not included in our initial model, it is nonetheless broadly consistent with the other approaches, identifying population isolation beginning between 10,000 and 15,000 generations ago, a clear domestication bottleneck, and ultimately rapid population expansion in maize to an extremely large extant size of $\approx 10^9$ (Figure S2).
Our assessment of the historical demography of maize and teosinte provides context for subsequent analyses of linked selection.


\begin{figure}[!tb]
  \begin{center}
  \includegraphics[width=.45\textwidth] {FigsAndFiles/Pi_and_Tajima.png}
\end{center}
\caption{\textbf{A} and \textbf{B} show mean pairwise diversity $\pi$, $\pm$ one standard deviation, while \textbf{C} and \textbf{D} depict and Tajima's D in 1kb windows from genic and nongenic regions of maize and teosinte.   \label{fig:diversity} }
\end{figure}


\begin{figure}[!tb]
%\vspace*{.05in}
\centering
\includegraphics[width=.5\textwidth]{FigsAndFiles/DomesticationModel/domesticationModel.pdf}
\caption{Parameter estimates for a basic bottleneck model of maize domestication. See methods for details. \label{fig:bottleneck} }
\end{figure}


\subsection*{Hard sweeps do not explain diversity differences} %JRI
When selection increases the frequency of a new beneficial mutation, a signature of reduced diversity is left at surrounding linked sites \cite{smith1974}.
To evaluate whether patterns of such ``hard sweeps'' could explain observed differences in diversity between genic and intergenic regions of the genome, we compared diversity around missense and synonymous substitutions between \emph{Tripsacum} and either maize or teosinte.
If a substantial proportion of missense mutations have been fixed due to hard sweeps, diversity around these substitutions should be lower than around synonymous substitutions. 
We observe this pattern around the causative amino acid substitution in the maize domestication locus \emph{tga1} (Figure S1), likely the result of a hard sweep during domestication \cite{wang2015}. Genome-wide, however, we observe no differences in diversity at sites near synonymous versus missense substitutions in either maize or teosinte (Figure \ref{fig:hardSweeps}).

%\mbh{I'm curious whether linked selection is affecting the diversity around a sizable proportion of synonymous substitutions.  Would it be worthwhile to look at diversity around the subset of synonymous substitutions that are the least linked to missense substitutions just to make sure it is not substantially higher? Or perhaps generate a plot of diversity around synonymous substitutions vs. distance of these subs from missense subs?}

\begin{figure*}[tb]
%\vspace*{.05in}
\centering
\includegraphics[width=.45\textwidth]{FigsAndFiles/plotDiversity_TvM_Folded2_Significance_Aug}
\hspace{0.05\textwidth} \includegraphics[width=.45\textwidth]{FigsAndFiles/plotDiversity_TvT_Folded2_Significance_Aug}
\caption{Pairwise diversity surrounding synonymous and missense  substitutions in {\bf A} maize and {\bf B} teosinte. Axes show absolute diversity values (right) and values relative to mean nucleotide diversity in windows $\geq 0.01 cM$ from a substitution (left).  Lines depict a loess curve (span of 0.01) and shading represents bootstrap-based 95\% confidence intervals. Inset plots depict a larger range on the x-axis. \label{fig:hardSweeps}}
\end{figure*}

Previous analyses have suggested that this approach may have limited power because a relatively high proportion of missense substitutions will be found in genes that, due to weak purifying selection, have higher genetic diversity \cite{enard2014}. 
To address this concern, we took advantage of genome-wide estimates of evolutionary constraint \cite{rodgers2015} calculated using genomic evolutionary rate profile (GERP) scores \cite{davydov2010}. 
We then evaluated substitutions only in subsets of genes in the highest and lowest 10\% quantile of mean GERP score, putatively representing genes under the strongest and weakest purifying selection. 
As expected, we see higher diversity around substitutions in genes under weak purifying selection, but we still find no difference in diversity near synonymous and missense substitutions in either subset of the data (Figure S3).
Taken together, these data suggest hard sweeps do not play a major role in patterning genic diversity in either maize or teosinte.

\subsection*{Diversity is strongly influenced by purifying selection} %JRI
In the case of purifying or background selection, diversity is reduced in functional regions of the genome via removal of deleterious mutations \cite{charlesworth1993}.
We investigated purifying selection in maize and teosinte by evaluating the reduction of diversity around genes.
Pairwise diversity is strongly reduced within genes for both maize and teosinte (Figure \ref{fig:purify}A) but recovers quickly at sites outside of genes, consistent with the low levels of linkage disequilibrium generally observed in these subspecies \cite{chia2012}. 
The reduction in relative diversity is more pronounced in teosinte, reaching lower levels in genes and occurring over a wider region.  

Our previous comparison of synonymous and missense substitutions has low power to detect the effects of selection acting on multiple beneficial mutations or standing genetic variation, because in such cases diversity around the substitution may be reduced to a lesser degree \cite{messer2013}. 
Nonetheless, such ``soft sweeps'' are still expected to occur more frequently in functional regions of the genome and could provide an alternative explanation to purifying selection for the observed reduction of diversity at linked sites in genes. 
To test this possibility, we performed a genome-wide scan for selection using the H12 statistic, a method expected to be sensitive to both hard and soft sweeps \cite{garud2015}. 
Qualitative differences between maize and teosinte in patterns of diversity within and outside of genes remained unchanged even after removing genes in the top 20\% quantile of H12 (Figure S7A).
%\mbh{This makes me wonder about the top 20\%. Is the pattern of diversity reduction different in this fraction of the genome?  If so, that seems like something that should be mentioned.}
%TB Says: Sites in the top 20% of H12 are unlikely to be under background selection, so maize vs teo background selection pattern at these sites seems strange to investigate.
We interpret these combined results as suggesting that purifying selection has predominantly shaped diversity near genes and left a more pronounced signature in the teosinte genome due to the increased efficacy of selection resulting from differences in long-term effective population size.

\begin{figure*}[!tb]
%\vspace*{.05in}
\centering
\includegraphics[width=.45\textwidth]{FigsAndFiles/distanceToGene_WithSignificance_Folded2_manuscript.png} \includegraphics[width=.45\textwidth]{FigsAndFiles/distanceToGene_WithSignificance_Singletons_Downsampled_threeLines_manuscript.png}
\caption{Relative diversity versus distance to nearest gene in maize and teosinte. 
Shown are \textbf{A} pairwise nucleotide diversity and \textbf{B} singleton diversity.  
Relative diversity is calculated compared to the mean diversity in windows $\geq 0.01 cM$ or $\geq 0.02 cM$ from the nearest
gene for pairwise diversity and singletons, respectively. 
  Lines depict cubic smoothing splines with smoothing parameters chosen via generalized cross validation and shading depicts bootstrap-based 95\% confidence intervals.
  Inset plots depict a smaller range on the x-axis. \label{fig:purify}
  }
\end{figure*}

\subsection*{Population expansion leads to stronger purifying selection in modern maize} %JRI
Motivated by the rapid post-domestication expansion of maize evident in our demographic analyses, we reasoned that low-frequency --- and thus younger --- polymorphisms might show patterns distinct from pairwise diversity, which is determined primarily by intermediate frequency --- therefore comparably older --- alleles. 
Singleton diversity around missense and synonymous substitutions (Figure S4) appears nearly identical to results from pairwise diversity (Figure \ref{fig:hardSweeps}), providing little support for a substantial recent increase in the number or strength of hard sweeps occurring in maize. 

In contrast, we observe a significant shift in the effects of purifying selection: singleton polymorphisms are more strongly reduced in and near genes in maize than in teosinte, even after downsampling our maize data to account for differences in sample size (Figure \ref{fig:purify}B). 
This result is the opposite of the pattern observed for $\pi$, where teosinte demonstrated a stronger reduction of diversity in and around genes than did maize. 
As before, this relationship remained after we removed the 20\% of genes with the highest H12 values (Figure S7).
While direct comparison of pairwise and singleton diversity within taxa is consistent with non-equilibrium dynamics in teosinte, these too reveal much stronger differences in maize (Figure S5) and mirror results from simulations of purifying selection (Figure S6).


\section*{DISCUSSION}


\subsection*{Demography of domestication} %JRI
Although a number of authors have investigated the demography of maize domestication \cite{eyre1998, tenaillon2004, wright2005}, these  efforts relied on data only from genic regions of the genome and made a number of limiting assumptions about the demographic model.  We show that diversity within genes has been strongly reduced by the effects of linked selection, such that even synonymous polymorphisms in genes are not representative of diversity at unconstrained sites. This implies that genic polymorphism data are unable to tell the complete or accurate demographic history of maize, but the rapid recovery of diversity outside of genes demonstrates that sites far from genes can be reasonably used for demographic inference. Furthermore, by utilizing the full joint SFS, we are able to estimate population growth, gene flow, and the strength of the domestication bottleneck without making assumptions about its duration.  
This model paves the way for future work on the demography of domestication, evaluating for example the significance of differences in gene flow estimated here or removing assumptions about demographic history in teosinte.

One surprising result from our model is the estimated divergence time of maize and teosinte approximately $15,000$ generations before present.
While this appears to conflict with archaeological estimates \cite{piperno2009}, we emphasize that this estimate reflects the fact that the genetic split between populations likely preceded anatomical changes that can be identified in the archaeological record. 
We also note that our result may be inflated due to population structure, as our geographically diverse sample of teosinte may include populations diverged from those that gave rise to maize.

The estimated bottleneck of $\approx 5\%$ of the ancestral teosinte population seems low given that maize landraces exhibit $\approx 80\%$ of the diversity of teosinte \cite{hufford2012}, but our model suggests that the effects of the bottleneck on diversity are likely ameliorated by both gene flow and rapid population growth (Figure \ref{fig:bottleneck}).   
Although we estimate that the modern effective size of maize is larger than teosinte, the small size of our sample reduces our power to identify the low frequency alleles most sensitive to rapid population growth \cite{keinan2012}, and our model is unable to incorporate growth faster than exponential.  
Both alternative approaches we employ estimate a much larger modern effective size of maize in the range of $\approx 10^6 - 10^9$, an order of magnitude or more than the current size of teosinte.
Census data suggest these estimates are plausible: there are 47.9 million ha of open-pollinated maize in production \cite{cimmyt1999}, likely planted at a density of $\approx 25,000$ individuals per hectare \cite{baden2001}.
Assuming the effective size is only $\approx 0.4\%$ of the census size (i.e. 1 ear for every 1000 male plants), this still implies a modern effective population size of more than four billion.
While these genetic and census estimates are likely inaccurate, all of the evidence points to the fact that the modern effective size of maize is extremely large.


\subsection*{Hard sweeps do not shape genome-wide diversity in maize} %JRI
Our findings demonstrate that classic hard selective sweeps have not contributed substantially to genome-wide patterns of diversity in maize, a result we show is robust to concerns about power due to the effects of purifying selection  \cite{enard2014}. 
Although our approach ignores the potential for hard sweeps in noncoding regions of the genome, a growing body of evidence argues against hard sweeps as the prevalent mode of selection shaping maize variability. 
Among well-characterized domestication loci, only the gene \emph{tga1} shows evidence of a hard sweep on a missense mutation \cite{wang2015}, while published data for several loci are consistent with soft sweeps from standing variation \cite{studer2011} or multiple mutations \cite{wills2013}. 
Moreover, genome-wide studies of domestication \cite{hufford2012},  local adaptation \cite{Takuno15062015} and modern breeding \cite{van2012historical, beissinger2014} all support the importance of standing variation as primary sources of adaptive variation. 
Soft sweeps are expected to be common when $2N_e\mu_b \ge 1$, where $\mu_b$ is the mutation rate of beneficial alleles with selection coefficient $s_b$ \cite{messer2013}. Assuming a mutation rate of $3 \times 10^{-8}$ \cite{clark2005} and that on the order of $\approx 1-5\%$ of mutations are beneficial \cite{eyre2007}, this implies that soft sweeps should be common in both maize and teosinte for mutational targets $>>10kb$ --- a plausible size for quantitative traits or for regulatory evolution targeting genes with large up- or down-stream control regions \cite[e.g.]{studer2011}.
Indeed, many adaptive traits in both maize \cite{wallace2014} and teosinte \cite{weber2008} are highly quantitative,  and adaptation in both maize \cite{hufford2012} and teosinte \cite{pyhajarvi2013} has involved selection on regulatory variation.

The absence of evidence for a genome-wide impact of hard sweeps in coding regions differs markedly from observations in \emph{Drosophila} \cite{sattath2011} and \emph{Capsella} \cite{williamson2014}, but is consistent with data from humans \cite{hernandez2011}.
Comparisons of the estimated percentages of nonsynonmyous substitutions fixed by natural selection \cite{ross2009,sella2009,eyre2009estimating,williamson2014} give similar results. 
While differences in long-term $N_e$ likely explains some of the observed variation across species, we see little change in the importance of hard sweeps in genes in singleton diversity in modern maize (Figure S4), perhaps suggesting other factors may contribute to these differences as well.  
One possibility, for example, is that, if mutational target size scales with genome size, the larger genomes of human and maize may offer more opportunities for noncoding loci to contribute to adaptation, with hard sweeps on nonsynonymous variants then playing a relatively smaller role. 
Support for this idea comes from numerous cases of adaptive transposable element insertion modifying gene regulation in maize \cite{studer2011,mao2015,yang2013} and studies of local adaptation that show enrichment for SNPs in regulatory regions in teosinte \cite{pyhajarvi2013} and humans \cite{fraser2013gene} but for nonsynonymous variants in the smaller \emph{Arabidopsis} genome \cite{hancock2011adaptation}.  
Our results, for example, are not dissimilar to findings in the comparably-sized mouse genome, where no differences are seen in diversity around nonsynonymous and synonymous substitutions in spite of a large $N_e$ and  as many as 80\% of adaptive substitutions occurring outside of genes\cite{Halligan:2013ir}.
Future comparative analyses using a common statistical framework (e.g. \cite{corbett2015} )and considering additional ecological and life history factors (c.f. \cite{leffler2012}) should allow explicit testing of this idea.

\subsection*{Demography influences the efficiency of purifying selection} 
One of our more striking findings is that the impact of purifying selection on maize and teosinte qualitatively changed over time. 
We observe a more pronounced decrease in $\pi$ around genes in teosinte than maize (Figure \ref{fig:purify}A), but the opposite trend when we evaluate diversity using singleton polymorphisms (Figure \ref{fig:purify}B).
The efficiency of purifying selection is proportional to effective population size \cite{kimura1984}, and these results are thus consistent with our demographic analyses which show a domestication bottleneck and smaller long-term $N_e$ in maize \cite{eyre1998, tenaillon2004, wright2005, ross2009} followed by recent rapid expansion and a much larger modern $N_e$. 
Simple foward-in-time population genetic simulations qualitatively confirm these results, and further suggest that the observed patterns are likely cause by sites under relatively weak purifying selection (Figure S6).


Although demographic change affects the efficiency of purifying selection, it may have limited implications for genetic load. 
Recent population bottlenecks and expansions have increased the relative abundance of rare and 
deleterious variants in domesticated plants \cite{Gunther:2010gt,renaut2015} and human populations out of Africa \cite{keinan2012,coventry2010}, and such variants may play an important role in phenotypic variation \cite{Mezmouk:2014es, coventry2010, eyre2010}.
Nonetheless, demographic history may have little impact on the overall genetic load of populations \cite{do2015,simons2014}, as decreases in $N_e$ that allow weakly deleterious variants to escape selection also help purge strongly deleterious ones, and the increase of new deleterious mutations in expanding populations is mitigated by their lower initial frequency and the increasing efficiency of purifying selection \cite{simons2014, lohmueller2014}. 

\section*{Rapid changes in linked selection} %JRI
Our results demonstrate that consideration of long-term differences in $N_e$ cannot fully capture the dynamic relationship between demography and selection. 
While a number of authors have tested for selection using methods that explicitly incorporate or are robust to demographic change \cite{eyre2009estimating, zeng2010} and others have compared estimates of the efficiency of adaptive and purifying selection across species \cite{popadin2012} or populations \cite{Elyashiv:2010ga}, previous analyses of the impact of linked selection on genome-wide diversity have relied on single estimates of the effective population size \cite{corbett2015,leffler2012}.
Our results show that demographic change over short periods of time can quickly change the dynamics of linked selection: mutations arising in extant maize populations are much more strongly impacted by the effects of selection on linked sites than would be suggested by analyses using long-term effective population size.
As many natural and domesticated populations have undergone considerable demographic change in their recent past, long-term comparisons of $N_e$ are likely not informative about current processes affecting allele frequency trajectories.

\section*{METHODS}
\subsection*{BASH, R, and Python scripts}
All scripts used for analysis are available in an online repository at \url{https://github.com/timbeissinger/Maize-Teo-Scripts}. 

\subsection*{Plant materials}
We made use of published sequences from inbred accessions of teosinte (Z. {\it mays} ssp. {\it parviglumis}) and maize landraces from the Maize HapMap3 panel as part of the Panzea project  \cite{chia2012, lemmon2014, bukowski2015}. 
From these data, we removed 4 teosinte individuals that were not ssp. \textit{parviglumis} or appeared as outliers in an initial principal component analysis conducted with the package adegenet \cite{jombart2011} (Figure S8), leaving 13 teosinte and 23 maize that were used for all subsequent analyses (Table S1). We also utilized a single individual of (\textit{Tripsacum dactyloides}) as an outgroup.  All bam files are available from Cyverse at \url{/iplant/home/shared/data_commons/curated/Beissinger_MaizeTeo_2016}. 

\subsection*{Physical and genetic maps}
Sequences were mapped to the maize B73 version 3 reference genome \cite{schnable2009} (\url{ftp://ftp.ensemblgenomes.org/pub/plants/release-22/fasta/zea\_mays/dna/}) as described by \cite{bukowski2015}. 
All analyses made use of uniquely mapping reads with mapping quality score $\geq  30$ and bases with base quality score $\geq 20$; quality scores around indels were adjusted following \cite{li2011statistical}.
We converted physical coordinates to genetic coordinates via linear interpolation of the previously published 1cM resolution NAM genetic map \cite{glaubitz2014}. 

\subsection*{Estimating the site frequency spectrum}
We estimated both the genome-wide site frequency spectrum (SFS) as well as a separate SFS for genic (within annotated transcript) and intergenic ($\geq 5kb$ from a transcript) regions. 
We used the biomaRt package \cite{durinck2009,durinck2005} of R \cite{R2014} to parse annotations from genebuild version 5b of AGPv3. 
We estimated single population and joint SFS with the software ANGSD \cite{korneliussen2014}, including all positions with at least one aligned read in $\geq 80\%$ of samples in one or both populations.
We assumed individuals were fully inbred and treated each line as a single haplotype. Because ANGSD cannot calculate a folded joint SFS, we first polarized SNPs using the maize reference genome and then folded spectra using $\delta$a$\delta$i \cite{gutenkunst2009}.

\subsection*{Demographic inference}
We used the software $\delta$a$\delta$i \cite{gutenkunst2009} to estimate parameters of a domestication bottleneck from the joint maize-teosinte SFS, using only sites $>5 kb$ from a gene to ameliorate the effects of linked selection.
To minimize the number of parameters estimated, we employed a simple demographic model which posits a teosinte population of constant effective size $N_a$.  At time $T_b$ generations in the past, this population gave rise to a maize population of size $N_b$ which grew exponentially to size $N_m$ in the present (Figure \ref{fig:bottleneck}).
The model includes migration of $M_{mt}$ individuals each generation from maize to teosinte and $M_{tm}$ individuals from teosinte to maize.  We estimated $N_a$ using $\delta$a$\delta$i's estimation of $\theta=4N_a\mu$ from the data and a mutation rate of $\mu = 3 \times 10^{-8}$ \cite{clark2005}. 
We estimated all other parameters using 1,000 $\delta$a$\delta$i optimizations and allowing initial values between runs to be randomly perturbed by a factor of 2.  
Optimized parameters along with their initial values and upper and lower bounds can be found in table S2. We report parameter estimates from the optimization run with the highest log-likelihood.

We further made use of a large genotyping data set of more than 4,000 partially imputed maize landraces \cite{Hearne2015} to estimate the modern maize $N_e$ from singleton counts. 
We filtered these data to include only SNPs with data in $\geq 1,500$ individuals, and then projected the SFS down to a sample of 500 individuals by sampling each marker without replacement 1,000 times according to the observed allele frequencies.
We then estimated $N_e$ from the data assuming $\mu = 3 \times 10^{-8}$ \cite{clark2005} and the relation  $4N_e\mu = \frac{S}{L}$ \cite{fu1993}, where $S$ is the total number of singleton SNPs and $L$ is the total number of SNPs in the dataset.

As a final estimate of demography, we employed MSMC \cite{schiffels2014}  to complement our model-based demographic inference. 
We used six each of maize and teosinte (BKN022, BKN025, BKN029, BKN030, BKN031, BKN033, TIL01, TIL03, TIL09, TIL10, TIL11 and TIL14), treating each inbred genome as a single haplotype.
We called SNPs in ANGSD \cite{korneliussen2014} using a SNP p-value of $1e-6$ against a reference genome masked using SNPable (\url{http://lh3lh3.users.sourceforge.net/snpable.shtml}). 
We then removed heterozygous genotypes and filtered sites with a mapping quality $<30$, a base quality $<20$, or a $|\textit{log}_2(\text{depth})|<1$.
We ran MSMC with pattern parameter $20\times2+20\times4+10\times2$ (Figure S2A) for population size inference. To estimate the rate of cross-coalescence we used four maize and four teosinte haplotypes with pattern parameter $20\times1 + 20\times2$ (Figure \S2B).

\subsection*{Diversity}
We made use of the software ANGSD \cite{korneliussen2014} for diversity calculations and genotype calling. 
We calculated diversity statistics in maize and teosinte in 1 kb non-overlapping windows using filters as described above for the SFS. 
We used allele counts to estimate the number of singleton polymorphisms in each window, and used binomial sampling to create a second maize data set down-sampled to have the same number of samples as teosinte.
We called genotypes in maize, teosinte, and \textit{ Tripsacum} at sites with a SNP p-value $<10^{-6}$ and when the genotype posterior probability $>0.95$. 
We identified substitutions in maize and teosinte as all sites with a fixed difference with \textit{Tripsacum} and $\leq 20\%$ missing data. 
Substitutions were classified as synonymous, or missense using the ensembl variant effects predictor \cite{mclaren2010}.
For each window with $\geq 100bp$ of data we computed the genetic distance between the window center and the nearest synonymous and missense substitution as well as the genetic distance to the center of the nearest gene transcript.  

\subsection*{Selection scan}
We scanned the genome to identify sites that have experienced recent positive selection using the H12 statistic \cite{garud2015} in sliding windows of 200 SNPs with a step of 25 SNPs.

\subsection*{Simulations}
We used the program \emph{bneck\_selection\_ind} included in version 0.4.4 of the forward-in-time population genetic simulation library \emph{fwdpp} \citep[\url{https://github.com/molpopgen/fwdpp}[]{thornton2014genetics}. 
All simulations used a population mutation rate of $\theta=20$, a population recombination rate of $\rho=20$, and simulated 150,000 burn-in generations at an ancestral population size of $N_1=15,000$ to establish equilibrium, after which the population instantly changed to size $N_2$ and then grew exponentially for 1,000 generations to size $N_3$. 
To simulate a constant size population emulating teosinte, we set $N_2=N_3=15,000$.
For maize we simulated a bottleneck similar to that estimated in Figure \ref{fig:bottleneck} by setting $N_2=750$,   followed by exponential growth to a large modern population size of $N_3=150,000$. 
For each taxon, we performed 1,000 simulations for each of five values of the strength of purifying selection: $s=\{0,10^{-6},10^{-5},10^{-4},10^{-3}\}$. 
All mutations were assumed to be codominant.
To mimic nonsynonymous changes at a coding locus, we assumed that $\frac{3}{4}$ of mutations were selected.
We calculated summary statistic across all sites using version 0.3.4 of \emph{msstats} (\url{https://github.com/molpopgen/msstats/releases}).

\section*{Correspondence}
Correspondence may be addressed to Tim Beissinger (beissingert@missouri.edu) or Jeffrey Ross-Ibarra (rossibarra@ucdavis.edu)

\section*{ACKNOWLEDGEMENTS}
We are indebted to Graham Coop and Simon Aeschbacher for their constructive input during this study. We thank Robert Bukowski and Qi Sun for providing early-access data from maize HapMap3. Funding was provided by NSF Plant Genome Research Project 1238014, the USDA-Agricultural Research Service, and USDA Hatch project CA-D-PLS-2066-H.

\section*{AUTHOR CONTRIBUTIONS}
TMB and JRI devised this study. TMB, LW, JRI, and KC analyzed the data. AD performed early-stage simulations. TMB, JRI, and MBH wrote the manuscript.

\section*{COMPETING INTERESTS STATEMENT}
The authors declare no competing financial interests.


%{\scriptsize \sf
%\renewcommand{\baselinestretch}{2.0}
%\bibliographystyle{naturemag}
%\bibliography{Reference}
%}

\renewcommand\refname{References (Main Text)}

\begin{thebibliography}{10}
\expandafter\ifx\csname url\endcsname\relax
  \def\url#1{\texttt{#1}}\fi
\expandafter\ifx\csname urlprefix\endcsname\relax\def\urlprefix{URL }\fi
\providecommand{\bibinfo}[2]{#2}
\providecommand{\eprint}[2][]{\url{#2}}

\bibitem{dobzhansky1957}
\bibinfo{author}{Dobzhansky, T.} \& \bibinfo{author}{Pavlovsky, O.}
\newblock \bibinfo{title}{An experimental study of interaction between genetic
  drift and natural selection}.
\newblock \emph{\bibinfo{journal}{Evolution}} \bibinfo{pages}{311--319}
  (\bibinfo{year}{1957}).

\bibitem{voight2005}
\bibinfo{author}{Voight, B.~F.} \emph{et~al.}
\newblock \bibinfo{title}{Interrogating multiple aspects of variation in a full
  resequencing data set to infer human population size changes}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences of the United States of America}} \textbf{\bibinfo{volume}{102}},
  \bibinfo{pages}{18508--18513} (\bibinfo{year}{2005}).

\bibitem{gutenkunst2009}
\bibinfo{author}{Gutenkunst, R.~N.}, \bibinfo{author}{Hernandez, R.~D.},
  \bibinfo{author}{Williamson, S.~H.} \& \bibinfo{author}{Bustamante, C.~D.}
\newblock \bibinfo{title}{Inferring the joint demographic history of multiple
  populations from multidimensional snp frequency data}.
\newblock \emph{\bibinfo{journal}{PLoS genetics}} \textbf{\bibinfo{volume}{5}},
  \bibinfo{pages}{e1000695} (\bibinfo{year}{2009}).

\bibitem{akey2009}
\bibinfo{author}{Akey, J.~M.}
\newblock \bibinfo{title}{Constructing genomic maps of positive selection in
  humans: Where do we go from here?}
\newblock \emph{\bibinfo{journal}{Genome research}}
  \textbf{\bibinfo{volume}{19}}, \bibinfo{pages}{711--722}
  (\bibinfo{year}{2009}).

\bibitem{smith1974}
\bibinfo{author}{Smith, J.~M.} \& \bibinfo{author}{Haigh, J.}
\newblock \bibinfo{title}{The hitch-hiking effect of a favourable gene}.
\newblock \emph{\bibinfo{journal}{Genetical research}}
  \textbf{\bibinfo{volume}{23}}, \bibinfo{pages}{23--35}
  (\bibinfo{year}{1974}).

\bibitem{slotte2014}
\bibinfo{author}{Slotte, T.}
\newblock \bibinfo{title}{The impact of linked selection on plant genomic
  variation}.
\newblock \emph{\bibinfo{journal}{Briefings in functional genomics}}
  \textbf{\bibinfo{volume}{13}}, \bibinfo{pages}{268--275}
  (\bibinfo{year}{2014}).

\bibitem{charlesworth1993}
\bibinfo{author}{Charlesworth, B.}, \bibinfo{author}{Morgan, M.} \&
  \bibinfo{author}{Charlesworth, D.}
\newblock \bibinfo{title}{The effect of deleterious mutations on neutral
  molecular variation.}
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{134}},
  \bibinfo{pages}{1289--1303} (\bibinfo{year}{1993}).

\bibitem{sella2009}
\bibinfo{author}{Sella, G.}, \bibinfo{author}{Petrov, D.~A.},
  \bibinfo{author}{Przeworski, M.} \& \bibinfo{author}{Andolfatto, P.}
\newblock \bibinfo{title}{Pervasive natural selection in the drosophila
  genome?}
\newblock \emph{\bibinfo{journal}{PLoS genetics}} \textbf{\bibinfo{volume}{5}},
  \bibinfo{pages}{e1000495} (\bibinfo{year}{2009}).

\bibitem{elyashiv2014}
\bibinfo{author}{Elyashiv, E.} \emph{et~al.}
\newblock \bibinfo{title}{A genomic map of the effects of linked selection in
  drosophila}.
\newblock \emph{\bibinfo{journal}{arXiv preprint arXiv:1408.5461}}
  (\bibinfo{year}{2014}).

\bibitem{andolfatto2005}
\bibinfo{author}{Andolfatto, P.}
\newblock \bibinfo{title}{Adaptive evolution of non-coding dna in drosophila}.
\newblock \emph{\bibinfo{journal}{Nature}} \textbf{\bibinfo{volume}{437}},
  \bibinfo{pages}{1149--1152} (\bibinfo{year}{2005}).

\bibitem{cutter2013}
\bibinfo{author}{Cutter, A.~D.} \& \bibinfo{author}{Payseur, B.~A.}
\newblock \bibinfo{title}{Genomic signatures of selection at linked sites:
  unifying the disparity among species}.
\newblock \emph{\bibinfo{journal}{Nature Reviews Genetics}}
  \textbf{\bibinfo{volume}{14}}, \bibinfo{pages}{262--274}
  (\bibinfo{year}{2013}).

\bibitem{corbett2015}
\bibinfo{author}{Corbett-Detig, R.~B.}, \bibinfo{author}{Hartl, D.~L.} \&
  \bibinfo{author}{Sackton, T.~B.}
\newblock \bibinfo{title}{Natural selection constrains neutral diversity across
  a wide range of species}.
\newblock \emph{\bibinfo{journal}{PLoS Biol}} \textbf{\bibinfo{volume}{13}},
  \bibinfo{pages}{e1002112} (\bibinfo{year}{2015}).
\newblock \urlprefix\url{http://dx.doi.org/10.1371%2Fjournal.pbio.1002112}.

\bibitem{leffler2012}
\bibinfo{author}{Leffler, E.~M.} \emph{et~al.}
\newblock \bibinfo{title}{Revisiting an old riddle: what determines genetic
  diversity levels within species}.
\newblock \emph{\bibinfo{journal}{PLoS Biol}} \textbf{\bibinfo{volume}{10}},
  \bibinfo{pages}{e1001388} (\bibinfo{year}{2012}).

\bibitem{duchen2013}
\bibinfo{author}{Duchen, P.}, \bibinfo{author}{{\v{Z}}ivkovi{\'c}, D.},
  \bibinfo{author}{Hutter, S.}, \bibinfo{author}{Stephan, W.} \&
  \bibinfo{author}{Laurent, S.}
\newblock \bibinfo{title}{Demographic inference reveals african and european
  admixture in the north american drosophila melanogaster population}.
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{193}},
  \bibinfo{pages}{291--301} (\bibinfo{year}{2013}).

\bibitem{reich1998}
\bibinfo{author}{Reich, D.~E.} \& \bibinfo{author}{Goldstein, D.~B.}
\newblock \bibinfo{title}{Genetic evidence for a paleolithic human population
  expansion in africa}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{95}}, \bibinfo{pages}{8119--8123}
  (\bibinfo{year}{1998}).

\bibitem{meyer2013}
\bibinfo{author}{Meyer, R.~S.} \& \bibinfo{author}{Purugganan, M.~D.}
\newblock \bibinfo{title}{Evolution of crop species: genetics of domestication
  and diversification}.
\newblock \emph{\bibinfo{journal}{Nature Reviews Genetics}}
  \textbf{\bibinfo{volume}{14}}, \bibinfo{pages}{840--852}
  (\bibinfo{year}{2013}).

\bibitem{ellegren2014}
\bibinfo{author}{Ellegren, H.}
\newblock \bibinfo{title}{Genome sequencing and population genomics in
  non-model organisms}.
\newblock \emph{\bibinfo{journal}{Trends in ecology \& evolution}}
  \textbf{\bibinfo{volume}{29}}, \bibinfo{pages}{51--63}
  (\bibinfo{year}{2014}).

\bibitem{matsuoka2002}
\bibinfo{author}{Matsuoka, Y.} \emph{et~al.}
\newblock \bibinfo{title}{A single domestication for maize shown by multilocus
  microsatellite genotyping}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{99}}, \bibinfo{pages}{6080--6084}
  (\bibinfo{year}{2002}).

\bibitem{wright2005}
\bibinfo{author}{Wright, S.~I.} \emph{et~al.}
\newblock \bibinfo{title}{The effects of artificial selection on the maize
  genome}.
\newblock \emph{\bibinfo{journal}{Science}} \textbf{\bibinfo{volume}{308}},
  \bibinfo{pages}{1310--1314} (\bibinfo{year}{2005}).

\bibitem{eyre1998}
\bibinfo{author}{Eyre-Walker, A.}, \bibinfo{author}{Gaut, R.~L.},
  \bibinfo{author}{Hilton, H.}, \bibinfo{author}{Feldman, D.~L.} \&
  \bibinfo{author}{Gaut, B.~S.}
\newblock \bibinfo{title}{Investigation of the bottleneck leading to the
  domestication of maize}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{95}}, \bibinfo{pages}{4441--4446}
  (\bibinfo{year}{1998}).

\bibitem{tenaillon2004}
\bibinfo{author}{Tenaillon, M.~I.}, \bibinfo{author}{U'Ren, J.},
  \bibinfo{author}{Tenaillon, O.} \& \bibinfo{author}{Gaut, B.~S.}
\newblock \bibinfo{title}{Selection versus demography: a multilocus
  investigation of the domestication process in maize}.
\newblock \emph{\bibinfo{journal}{Molecular Biology and Evolution}}
  \textbf{\bibinfo{volume}{21}}, \bibinfo{pages}{1214--1225}
  (\bibinfo{year}{2004}).

\bibitem{chia2012}
\bibinfo{author}{Chia, J.-M.} \emph{et~al.}
\newblock \bibinfo{title}{Maize hapmap2 identifies extant variation from a
  genome in flux}.
\newblock \emph{\bibinfo{journal}{Nature genetics}}
  \textbf{\bibinfo{volume}{44}}, \bibinfo{pages}{803--807}
  (\bibinfo{year}{2012}).

\bibitem{bukowski2015}
\bibinfo{author}{Bukowski, R.} \emph{et~al.}
\newblock \bibinfo{title}{Construction of the third generation zea mays
  haplotype map}.
\newblock \emph{\bibinfo{journal}{bioRxiv}} \bibinfo{pages}{026963}
  (\bibinfo{year}{2015}).

\bibitem{hufford2012}
\bibinfo{author}{Hufford, M.~B.} \emph{et~al.}
\newblock \bibinfo{title}{Comparative population genomics of maize
  domestication and improvement}.
\newblock \emph{\bibinfo{journal}{Nature genetics}}
  \textbf{\bibinfo{volume}{44}}, \bibinfo{pages}{808--811}
  (\bibinfo{year}{2012}).

\bibitem{ewing2016}
\bibinfo{author}{Ewing, G.~B.} \& \bibinfo{author}{Jensen, J.~D.}
\newblock \bibinfo{title}{The consequences of not accounting for background
  selection in demographic inference}.
\newblock \emph{\bibinfo{journal}{Molecular ecology}}
  \textbf{\bibinfo{volume}{25}}, \bibinfo{pages}{135--141}
  (\bibinfo{year}{2016}).

\bibitem{Hearne2015}
\bibinfo{author}{Hearne, S.}, \bibinfo{author}{Chen, C.},
  \bibinfo{author}{Buckler, E.} \& \bibinfo{author}{Mitchell, S.}
\newblock \bibinfo{title}{Unimputed gbs derived snps for maize landrace
  accessions represented in the seed-maize gwas panel}.
\newblock \bibinfo{howpublished}{\url{http://hdl.handle.net/11529/10034}}
  (\bibinfo{year}{2015}).
\newblock \bibinfo{note}{Accessed: 2015-02-16}.

\bibitem{keinan2012}
\bibinfo{author}{Keinan, A.} \& \bibinfo{author}{Clark, A.~G.}
\newblock \bibinfo{title}{Recent explosive human population growth has resulted
  in an excess of rare genetic variants}.
\newblock \emph{\bibinfo{journal}{science}} \textbf{\bibinfo{volume}{336}},
  \bibinfo{pages}{740--743} (\bibinfo{year}{2012}).

\bibitem{fu1993}
\bibinfo{author}{Fu, Y.-X.} \& \bibinfo{author}{Li, W.-H.}
\newblock \bibinfo{title}{Statistical tests of neutrality of mutations.}
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{133}},
  \bibinfo{pages}{693--709} (\bibinfo{year}{1993}).

\bibitem{clark2005}
\bibinfo{author}{Clark, R.~M.}, \bibinfo{author}{Tavar{\'e}, S.} \&
  \bibinfo{author}{Doebley, J.}
\newblock \bibinfo{title}{Estimating a nucleotide substitution rate for maize
  from polymorphism at a major domestication locus}.
\newblock \emph{\bibinfo{journal}{Molecular biology and evolution}}
  \textbf{\bibinfo{volume}{22}}, \bibinfo{pages}{2304--2312}
  (\bibinfo{year}{2005}).

\bibitem{schiffels2014}
\bibinfo{author}{Schiffels, S.} \& \bibinfo{author}{Durbin, R.}
\newblock \bibinfo{title}{Inferring human population size and separation
  history from multiple genome sequences}.
\newblock \emph{\bibinfo{journal}{Nature genetics}}  (\bibinfo{year}{2014}).

\bibitem{wang2015}
\bibinfo{author}{Wang, H.}, \bibinfo{author}{Studer, A.~J.},
  \bibinfo{author}{Zhao, Q.}, \bibinfo{author}{Meeley, R.} \&
  \bibinfo{author}{Doebley, J.~F.}
\newblock \bibinfo{title}{Evidence that the origin of naked kernels during
  maize domestication was caused by a single amino acid substitution in tga1}.
\newblock \emph{\bibinfo{journal}{Genetics}} \bibinfo{pages}{genetics--115}
  (\bibinfo{year}{2015}).

\bibitem{enard2014}
\bibinfo{author}{Enard, D.}, \bibinfo{author}{Messer, P.~W.} \&
  \bibinfo{author}{Petrov, D.~A.}
\newblock \bibinfo{title}{Genome-wide signals of positive selection in human
  evolution}.
\newblock \emph{\bibinfo{journal}{Genome research}}
  \textbf{\bibinfo{volume}{24}}, \bibinfo{pages}{885--895}
  (\bibinfo{year}{2014}).

\bibitem{rodgers2015}
\bibinfo{author}{Rodgers-Melnick, E.} \emph{et~al.}
\newblock \bibinfo{title}{Recombination in diverse maize is stable,
  predictable, and associated with genetic load}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{112}}, \bibinfo{pages}{3823--3828}
  (\bibinfo{year}{2015}).

\bibitem{davydov2010}
\bibinfo{author}{Davydov, E.~V.} \emph{et~al.}
\newblock \bibinfo{title}{Identifying a high fraction of the human genome to be
  under selective constraint using gerp++}.
\newblock \emph{\bibinfo{journal}{PLoS Comput Biol}}
  \textbf{\bibinfo{volume}{6}}, \bibinfo{pages}{e1001025}
  (\bibinfo{year}{2010}).

\bibitem{messer2013}
\bibinfo{author}{Messer, P.~W.} \& \bibinfo{author}{Petrov, D.~A.}
\newblock \bibinfo{title}{Population genomics of rapid adaptation by soft
  selective sweeps}.
\newblock \emph{\bibinfo{journal}{Trends in ecology \& evolution}}
  \textbf{\bibinfo{volume}{28}}, \bibinfo{pages}{659--669}
  (\bibinfo{year}{2013}).

\bibitem{garud2015}
\bibinfo{author}{Garud, N.~R.}, \bibinfo{author}{Messer, P.~W.},
  \bibinfo{author}{Buzbas, E.~O.} \& \bibinfo{author}{Petrov, D.~A.}
\newblock \bibinfo{title}{Recent selective sweeps in north american drosophila
  melanogaster show signatures of soft sweeps}.
\newblock \emph{\bibinfo{journal}{PLoS genetics}}
  \textbf{\bibinfo{volume}{11}}, \bibinfo{pages}{e1005004}
  (\bibinfo{year}{2015}).

\bibitem{piperno2009}
\bibinfo{author}{Piperno, D.~R.}, \bibinfo{author}{Ranere, A.~J.},
  \bibinfo{author}{Holst, I.}, \bibinfo{author}{Iriarte, J.} \&
  \bibinfo{author}{Dickau, R.}
\newblock \bibinfo{title}{Starch grain and phytolith evidence for early ninth
  millennium bp maize from the central balsas river valley, mexico}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{106}}, \bibinfo{pages}{5019--5024}
  (\bibinfo{year}{2009}).

\bibitem{cimmyt1999}
\bibinfo{author}{Program, T.~M.}
\newblock \emph{\bibinfo{title}{Development, maintenance, and seed
  multiplication of open-pollinated maize varieties}}
  (\bibinfo{publisher}{CIMMYT}, \bibinfo{address}{Mexico, D.F.},
  \bibinfo{year}{1999}), \bibinfo{edition}{2} edn.

\bibitem{baden2001}
\bibinfo{author}{Baden, W.~W.} \& \bibinfo{author}{Beekman, C.~S.}
\newblock \bibinfo{title}{Culture and agriculture: A comment on sissel
  schroeder, maize productivity in the eastern woodlands and great plains of
  north america}.
\newblock \emph{\bibinfo{journal}{American Antiquity}}
  \bibinfo{pages}{505--515} (\bibinfo{year}{2001}).

\bibitem{studer2011}
\bibinfo{author}{Studer, A.}, \bibinfo{author}{Zhao, Q.},
  \bibinfo{author}{Ross-Ibarra, J.} \& \bibinfo{author}{Doebley, J.}
\newblock \bibinfo{title}{Identification of a functional transposon insertion
  in the maize domestication gene tb1}.
\newblock \emph{\bibinfo{journal}{Nature genetics}}
  \textbf{\bibinfo{volume}{43}}, \bibinfo{pages}{1160--1163}
  (\bibinfo{year}{2011}).

\bibitem{wills2013}
\bibinfo{author}{Wills, D.~M.} \emph{et~al.}
\newblock \bibinfo{title}{From many, one: genetic control of prolificacy during
  maize domestication}.
\newblock \emph{\bibinfo{journal}{PLoS Genet}} \textbf{\bibinfo{volume}{9}},
  \bibinfo{pages}{e1003604} (\bibinfo{year}{2013}).

\bibitem{Takuno15062015}
\bibinfo{author}{Takuno, S.} \emph{et~al.}
\newblock \bibinfo{title}{Independent molecular basis of convergent highland
  adaptation in maize}.
\newblock \emph{\bibinfo{journal}{Genetics}}  (\bibinfo{year}{2015}).
\newblock
  \urlprefix\url{http://www.genetics.org/content/early/2015/06/15/genetics.115.178327.abstract}.
\newblock
  \eprint{http://www.genetics.org/content/early/2015/06/15/genetics.115.178327.full.pdf+html}.

\bibitem{van2012historical}
\bibinfo{author}{van Heerwaarden, J.}, \bibinfo{author}{Hufford, M.~B.} \&
  \bibinfo{author}{Ross-Ibarra, J.}
\newblock \bibinfo{title}{Historical genomics of north american maize}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{109}}, \bibinfo{pages}{12420--12425}
  (\bibinfo{year}{2012}).

\bibitem{beissinger2014}
\bibinfo{author}{Beissinger, T.~M.} \emph{et~al.}
\newblock \bibinfo{title}{A genome-wide scan for evidence of selection in a
  maize population under long-term artificial selection for ear number}.
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{196}},
  \bibinfo{pages}{829--840} (\bibinfo{year}{2014}).

\bibitem{eyre2007}
\bibinfo{author}{Eyre-Walker, A.} \& \bibinfo{author}{Keightley, P.~D.}
\newblock \bibinfo{title}{The distribution of fitness effects of new
  mutations}.
\newblock \emph{\bibinfo{journal}{Nature Reviews Genetics}}
  \textbf{\bibinfo{volume}{8}}, \bibinfo{pages}{610--618}
  (\bibinfo{year}{2007}).

\bibitem{wallace2014}
\bibinfo{author}{Wallace, J.}, \bibinfo{author}{Larsson, S.} \&
  \bibinfo{author}{Buckler, E.}
\newblock \bibinfo{title}{Entering the second century of maize quantitative
  genetics}.
\newblock \emph{\bibinfo{journal}{Heredity}} \textbf{\bibinfo{volume}{112}},
  \bibinfo{pages}{30--38} (\bibinfo{year}{2014}).

\bibitem{weber2008}
\bibinfo{author}{Weber, A.~L.} \emph{et~al.}
\newblock \bibinfo{title}{The genetic architecture of complex traits in
  teosinte (zea mays ssp. parviglumis): new evidence from association mapping}.
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{180}},
  \bibinfo{pages}{1221--1232} (\bibinfo{year}{2008}).

\bibitem{pyhajarvi2013}
\bibinfo{author}{Pyh{\"a}j{\"a}rvi, T.}, \bibinfo{author}{Hufford, M.~B.},
  \bibinfo{author}{Mezmouk, S.} \& \bibinfo{author}{Ross-Ibarra, J.}
\newblock \bibinfo{title}{Complex patterns of local adaptation in teosinte}.
\newblock \emph{\bibinfo{journal}{Genome biology and evolution}}
  \textbf{\bibinfo{volume}{5}}, \bibinfo{pages}{1594--1609}
  (\bibinfo{year}{2013}).

\bibitem{sattath2011}
\bibinfo{author}{Sattath, S.}, \bibinfo{author}{Elyashiv, E.},
  \bibinfo{author}{Kolodny, O.}, \bibinfo{author}{Rinott, Y.} \&
  \bibinfo{author}{Sella, G.}
\newblock \bibinfo{title}{Pervasive adaptive protein evolution apparent in
  diversity patterns around amino acid substitutions in drosophila simulans}.
\newblock \emph{\bibinfo{journal}{PLoS genetics}} \textbf{\bibinfo{volume}{7}},
  \bibinfo{pages}{e1001302} (\bibinfo{year}{2011}).

\bibitem{williamson2014}
\bibinfo{author}{Williamson, R.} \emph{et~al.}
\newblock \bibinfo{title}{Evidence for widespread positive and negative
  selection in coding and conserved noncoding regions of capsella grandiflora.}
\newblock \emph{\bibinfo{journal}{PLoS genetics}}
  \textbf{\bibinfo{volume}{10}}, \bibinfo{pages}{e1004622--e1004622}
  (\bibinfo{year}{2014}).

\bibitem{hernandez2011}
\bibinfo{author}{Hernandez, R.~D.} \emph{et~al.}
\newblock \bibinfo{title}{Classic selective sweeps were rare in recent human
  evolution}.
\newblock \emph{\bibinfo{journal}{science}} \textbf{\bibinfo{volume}{331}},
  \bibinfo{pages}{920--924} (\bibinfo{year}{2011}).

\bibitem{ross2009}
\bibinfo{author}{Ross-Ibarra, J.}, \bibinfo{author}{Tenaillon, M.} \&
  \bibinfo{author}{Gaut, B.~S.}
\newblock \bibinfo{title}{Historical divergence and gene flow in the genus
  zea}.
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{181}},
  \bibinfo{pages}{1399--1413} (\bibinfo{year}{2009}).

\bibitem{eyre2009estimating}
\bibinfo{author}{Eyre-Walker, A.} \& \bibinfo{author}{Keightley, P.~D.}
\newblock \bibinfo{title}{Estimating the rate of adaptive molecular evolution
  in the presence of slightly deleterious mutations and population size
  change}.
\newblock \emph{\bibinfo{journal}{Molecular biology and evolution}}
  \textbf{\bibinfo{volume}{26}}, \bibinfo{pages}{2097--2108}
  (\bibinfo{year}{2009}).

\bibitem{mao2015}
\bibinfo{author}{Mao, H.} \emph{et~al.}
\newblock \bibinfo{title}{A transposable element in a nac gene is associated
  with drought tolerance in maize seedlings}.
\newblock \emph{\bibinfo{journal}{Nature Communications}}
  \textbf{\bibinfo{volume}{6}} (\bibinfo{year}{2015}).

\bibitem{yang2013}
\bibinfo{author}{Yang, Q.} \emph{et~al.}
\newblock \bibinfo{title}{{CACTA}-like transposable element in {ZmCCT}
  attenuated photoperiod sensitivity and accelerated the postdomestication
  spread of maize}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{110}}, \bibinfo{pages}{16969--16974}
  (\bibinfo{year}{2013}).

\bibitem{fraser2013gene}
\bibinfo{author}{Fraser, H.~B.}
\newblock \bibinfo{title}{Gene expression drives local adaptation in humans}.
\newblock \emph{\bibinfo{journal}{Genome research}}
  \textbf{\bibinfo{volume}{23}}, \bibinfo{pages}{1089--1096}
  (\bibinfo{year}{2013}).

\bibitem{hancock2011adaptation}
\bibinfo{author}{Hancock, A.~M.} \emph{et~al.}
\newblock \bibinfo{title}{Adaptation to climate across the arabidopsis thaliana
  genome}.
\newblock \emph{\bibinfo{journal}{Science}} \textbf{\bibinfo{volume}{334}},
  \bibinfo{pages}{83--86} (\bibinfo{year}{2011}).

\bibitem{Halligan:2013ir}
\bibinfo{author}{Halligan, D.~L.} \emph{et~al.}
\newblock \bibinfo{title}{{Contributions of Protein-Coding and Regulatory
  Change to Adaptive Molecular Evolution in Murid Rodents}}.
\newblock \emph{\bibinfo{journal}{PLoS Genetics}} \textbf{\bibinfo{volume}{9}},
  \bibinfo{pages}{e1003995--14} (\bibinfo{year}{2013}).

\bibitem{kimura1984}
\bibinfo{author}{Kimura, M.}
\newblock \emph{\bibinfo{title}{The neutral theory of molecular evolution}}
  (\bibinfo{publisher}{Cambridge University Press}, \bibinfo{year}{1984}).

\bibitem{Gunther:2010gt}
\bibinfo{author}{G{\"u}nther, T.} \& \bibinfo{author}{Schmid, K.~J.}
\newblock \bibinfo{title}{{Deleterious amino acid polymorphisms in Arabidopsis
  thaliana and rice}}.
\newblock \emph{\bibinfo{journal}{Theoretical and Applied Genetics}}
  \textbf{\bibinfo{volume}{121}}, \bibinfo{pages}{157--168}
  (\bibinfo{year}{2010}).

\bibitem{renaut2015}
\bibinfo{author}{Renaut, S.} \& \bibinfo{author}{Rieseberg, L.~H.}
\newblock \bibinfo{title}{The accumulation of deleterious mutations as a
  consequence of domestication and improvement in sunflowers and other
  compositae crops}.
\newblock \emph{\bibinfo{journal}{Molecular biology and evolution}}
  \bibinfo{pages}{msv106} (\bibinfo{year}{2015}).

\bibitem{coventry2010}
\bibinfo{author}{Coventry, A.} \emph{et~al.}
\newblock \bibinfo{title}{Deep resequencing reveals excess rare recent variants
  consistent with explosive population growth}.
\newblock \emph{\bibinfo{journal}{Nature communications}}
  \textbf{\bibinfo{volume}{1}}, \bibinfo{pages}{131} (\bibinfo{year}{2010}).

\bibitem{Mezmouk:2014es}
\bibinfo{author}{Mezmouk, S.} \& \bibinfo{author}{Ross-Ibarra, J.}
\newblock \bibinfo{title}{{The pattern and distribution of deleterious
  mutations in maize.}}
\newblock \emph{\bibinfo{journal}{G3 (Bethesda, Md.)}}
  \textbf{\bibinfo{volume}{4}}, \bibinfo{pages}{163--171}
  (\bibinfo{year}{2014}).

\bibitem{eyre2010}
\bibinfo{author}{Eyre-Walker, A.}
\newblock \bibinfo{title}{Genetic architecture of a complex trait and its
  implications for fitness and genome-wide association studies}.
\newblock \emph{\bibinfo{journal}{Proceedings of the National Academy of
  Sciences}} \textbf{\bibinfo{volume}{107}}, \bibinfo{pages}{1752--1756}
  (\bibinfo{year}{2010}).

\bibitem{do2015}
\bibinfo{author}{Do, R.} \emph{et~al.}
\newblock \bibinfo{title}{No evidence that selection has been less effective at
  removing deleterious mutations in europeans than in africans}.
\newblock \emph{\bibinfo{journal}{Nature genetics}}
  \textbf{\bibinfo{volume}{47}}, \bibinfo{pages}{126--131}
  (\bibinfo{year}{2015}).

\bibitem{simons2014}
\bibinfo{author}{Simons, Y.~B.}, \bibinfo{author}{Turchin, M.~C.},
  \bibinfo{author}{Pritchard, J.~K.} \& \bibinfo{author}{Sella, G.}
\newblock \bibinfo{title}{The deleterious mutation load is insensitive to
  recent population history}.
\newblock \emph{\bibinfo{journal}{Nature genetics}}
  \textbf{\bibinfo{volume}{46}}, \bibinfo{pages}{220--224}
  (\bibinfo{year}{2014}).

\bibitem{lohmueller2014}
\bibinfo{author}{Lohmueller, K.~E.}
\newblock \bibinfo{title}{The impact of population demography and selection on
  the genetic architecture of complex traits}.
\newblock \emph{\bibinfo{journal}{PLoS Genetics}} \textbf{\bibinfo{volume}{10}}
  (\bibinfo{year}{2014}).

\bibitem{zeng2010}
\bibinfo{author}{Zeng, K.} \& \bibinfo{author}{Charlesworth, B.}
\newblock \bibinfo{title}{The effects of demography and linkage on the
  estimation of selection and mutation parameters}.
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{186}},
  \bibinfo{pages}{1411--1424} (\bibinfo{year}{2010}).

\bibitem{popadin2012}
\bibinfo{author}{Popadin, K.~Y.}, \bibinfo{author}{Nikolaev, S.~I.},
  \bibinfo{author}{Junier, T.}, \bibinfo{author}{Baranova, M.} \&
  \bibinfo{author}{Antonarakis, S.~E.}
\newblock \bibinfo{title}{Purifying selection in mammalian mitochondrial
  protein-coding genes is highly effective and congruent with evolution of
  nuclear genes}.
\newblock \emph{\bibinfo{journal}{Molecular biology and evolution}}
  \bibinfo{pages}{mss219} (\bibinfo{year}{2012}).

\bibitem{Elyashiv:2010ga}
\bibinfo{author}{Elyashiv, E.} \emph{et~al.}
\newblock \bibinfo{title}{{Shifts in the intensity of purifying selection: An
  analysis of genome-wide polymorphism data from two closely related yeast
  species}}.
\newblock \emph{\bibinfo{journal}{Genome Research}}
  \textbf{\bibinfo{volume}{20}}, \bibinfo{pages}{1558--1573}
  (\bibinfo{year}{2010}).


  \setcounter{firstbib}{\value{NAT@ctr}}
  
\end{thebibliography}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\renewcommand\refname{References (Methods)}

  \begin{thebibliography}{10}
\setcounter{NAT@ctr}{\value{firstbib}}
    \expandafter\ifx\csname url\endcsname\relax
  \def\url#1{\texttt{#1}}\fi
\expandafter\ifx\csname urlprefix\endcsname\relax\def\urlprefix{URL }\fi
\providecommand{\bibinfo}[2]{#2}
\providecommand{\eprint}[2][]{\url{#2}}



\bibitem{lemmon2014}
\bibinfo{author}{Lemmon, Z.~H.}, \bibinfo{author}{Bukowski, R.},
  \bibinfo{author}{Sun, Q.} \& \bibinfo{author}{Doebley, J.~F.}
\newblock \bibinfo{title}{The role of cis regulatory evolution
  in maize domestication}.
\newblock \emph{\bibinfo{journal}{PLoS Genet}} \textbf{\bibinfo{volume}{10}},
  \bibinfo{pages}{e1004745} (\bibinfo{year}{2014}).
\newblock \urlprefix\url{http://dx.doi.org/10.1371%2Fjournal.pgen.1004745}.

\bibitem{jombart2011}
\bibinfo{author}{Jombart, T.} \& \bibinfo{author}{Ahmed, I.}
\newblock \bibinfo{title}{adegenet 1.3-1: new tools for the analysis of
  genome-wide snp data}.
\newblock \emph{\bibinfo{journal}{Bioinformatics}}
  \textbf{\bibinfo{volume}{27}}, \bibinfo{pages}{3070--3071}
  (\bibinfo{year}{2011}).

\bibitem{schnable2009}
\bibinfo{author}{Schnable, P.~S.} \emph{et~al.}
\newblock \bibinfo{title}{The b73 maize genome: complexity, diversity, and
  dynamics}.
\newblock \emph{\bibinfo{journal}{science}} \textbf{\bibinfo{volume}{326}},
  \bibinfo{pages}{1112--1115} (\bibinfo{year}{2009}).

\bibitem{li2011statistical}
\bibinfo{author}{Li, H.}
\newblock \bibinfo{title}{A statistical framework for snp calling, mutation
  discovery, association mapping and population genetical parameter estimation
  from sequencing data}.
\newblock \emph{\bibinfo{journal}{Bioinformatics}}
  \textbf{\bibinfo{volume}{27}}, \bibinfo{pages}{2987--2993}
  (\bibinfo{year}{2011}).

\bibitem{glaubitz2014}
\bibinfo{author}{Glaubitz, J.~C.} \emph{et~al.}
\newblock \bibinfo{title}{Tassel-gbs: a high capacity genotyping by sequencing
  analysis pipeline}.
\newblock \emph{\bibinfo{journal}{PLoS One}} \textbf{\bibinfo{volume}{9}},
  \bibinfo{pages}{E90346} (\bibinfo{year}{2014}).

\bibitem{durinck2009}
\bibinfo{author}{Durinck, S.}, \bibinfo{author}{Spellman, P.~T.},
  \bibinfo{author}{Birney, E.} \& \bibinfo{author}{Huber, W.}
\newblock \bibinfo{title}{Mapping identifiers for the integration of genomic
  datasets with the r/bioconductor package biomart}.
\newblock \emph{\bibinfo{journal}{Nature protocols}}
  \textbf{\bibinfo{volume}{4}}, \bibinfo{pages}{1184--1191}
  (\bibinfo{year}{2009}).

\bibitem{durinck2005}
\bibinfo{author}{Durinck, S.} \emph{et~al.}
\newblock \bibinfo{title}{Biomart and bioconductor: a powerful link between
  biological databases and microarray data analysis}.
\newblock \emph{\bibinfo{journal}{Bioinformatics}}
  \textbf{\bibinfo{volume}{21}}, \bibinfo{pages}{3439--3440}
  (\bibinfo{year}{2005}).

\bibitem{R2014}
\bibinfo{author}{{R Core Team}}.
\newblock \emph{\bibinfo{title}{R: A Language and Environment for Statistical
  Computing}}.
\newblock \bibinfo{organization}{R Foundation for Statistical Computing},
  \bibinfo{address}{Vienna, Austria} (\bibinfo{year}{2014}).
\newblock \urlprefix\url{http://www.R-project.org/}.

\bibitem{korneliussen2014}
\bibinfo{author}{Korneliussen, T.~S.}, \bibinfo{author}{Albrechtsen, A.} \&
  \bibinfo{author}{Nielsen, R.}
\newblock \bibinfo{title}{Angsd: analysis of next generation sequencing data}.
\newblock \emph{\bibinfo{journal}{BMC bioinformatics}}
  \textbf{\bibinfo{volume}{15}}, \bibinfo{pages}{356} (\bibinfo{year}{2014}).

\bibitem{mclaren2010}
\bibinfo{author}{McLaren, W.} \emph{et~al.}
\newblock \bibinfo{title}{Deriving the consequences of genomic variants with
  the ensembl api and snp effect predictor}.
\newblock \emph{\bibinfo{journal}{Bioinformatics}}
  \textbf{\bibinfo{volume}{26}}, \bibinfo{pages}{2069--2070}
  (\bibinfo{year}{2010}).

\bibitem{thornton2014genetics}
\bibinfo{author}{Thornton, K.~R.}
\newblock \bibinfo{title}{A c++ template library for efficient forward-time
  population genetic simulation of large populations}.
\newblock \emph{\bibinfo{journal}{Genetics}} \textbf{\bibinfo{volume}{198}},
  \bibinfo{pages}{157--166} (\bibinfo{year}{2014}).

\end{thebibliography}


\end{document}
